{
 "metadata": {
  "name": "",
  "signature": "sha256:465062ea9607706d47f8cbd1121846f71514e7f52c1790dcf96ce9d5f06730ce"
 },
 "nbformat": 3,
 "nbformat_minor": 0,
 "worksheets": [
  {
   "cells": [
    {
     "cell_type": "markdown",
     "metadata": {},
     "source": [
      "# Preping reads for assembly\n",
      "\n",
      "From all the reads, produce 4 sets of files from trimmed reads:\n",
      "\n",
      " - PE, stranded libraries (left.fastq, right.fastq)\n",
      " - PE, long insert, non-stranded libraries (left.long.fastq, right.long.fastq)\n",
      " - SE, non-stranded libaries (all.se.fastq)\n",
      " - All libraries represented as a single reads in a fasta file for inchworm k-mer saturation analysis"
     ]
    },
    {
     "cell_type": "code",
     "collapsed": false,
     "input": [
      "import gzip, os, sys\n",
      "#import custom functions for displaying tables, bash commands\n",
      "sys.path.append(os.path.abspath(\"/home/damian/\"))\n",
      "from dk_ipython import *\n",
      "from IPython.display import HTML\n",
      "import json"
     ],
     "language": "python",
     "metadata": {},
     "outputs": [],
     "prompt_number": 2
    },
    {
     "cell_type": "code",
     "collapsed": false,
     "input": [
      "import json\n",
      "trimmLog = open('resource/trimPE.err')\n",
      "libs = []\n",
      "for line in trimmLog:\n",
      "    if line.find('Started with arguments') != -1:\n",
      "        #exclude the unstranded A1, A2, A1350, A2340 libraries\n",
      "        if line.find('_A') == -1:\n",
      "            libs.append(line.strip().split('.fastq.gz')[0].split()[-1].split('/')[-1][:-1])\n",
      "\n",
      "prefix = '/mlig_raw/trim/'\n",
      "f = [prefix + k + '1.fastq.gz.paired.trimmed.fastq.gz' for k in libs]\n",
      "r = [prefix + k + '2.fastq.gz.paired.trimmed.fastq.gz' for k in libs]\n",
      "\n",
      "HTML(textarea('Script for PE, stranded fastq.gz','''import gzip\n",
      "pairA = ''' + json.dumps(f) + '''\n",
      "pairB = ''' + json.dumps(r) + '''\n",
      "\n",
      "outA = open(\"left.stranded.fastq\",'w')\n",
      "outB = open(\"right.stranded.fastq\",'w')\n",
      "\n",
      "aCount = 0\n",
      "for a in pairA:\n",
      "    name = a.split('/')[-1].split('.')[0][:-3]\n",
      "\n",
      "    stream = gzip.open(a,'rb')\n",
      "    for line in stream:\n",
      "        seq = stream.next()\n",
      "        stream.next()\n",
      "        qual = stream.next()\n",
      "        outA.write('@r' + str(aCount).zfill(10) + '_' + name + '/1\\\\n' + seq + '+\\\\n' + qual)\n",
      "        aCount += 1\n",
      "\n",
      "bCount = 0\n",
      "for b in pairB:\n",
      "    name = b.split('/')[-1].split('.')[0][:-3]\n",
      "\n",
      "    stream = gzip.open(b,'rb')\n",
      "    for line in stream:\n",
      "        seq = stream.next()\n",
      "        stream.next()\n",
      "        qual = stream.next()\n",
      "        outB.write('@r' + str(bCount).zfill(10) + '_' + name + '/2\\\\n' + seq + '+\\\\n' + qual)\n",
      "        bCount += 1\n",
      "\n",
      "outA.close()\n",
      "outB.close()'''))"
     ],
     "language": "python",
     "metadata": {},
     "outputs": [
      {
       "html": [
        "<b>Script for PE, stranded fastq.gz</b>:<br>\n",
        "<textarea style=\"width: 1000px; height:200px\">\n",
        "import gzip\n",
        "pairA = [\"/mlig_raw/trim/mlig_pilot_reg_096h_R1.fastq.gz.paired.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_pilot_reg_072h_R1.fastq.gz.paired.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_pilot_reg_048h_R1.fastq.gz.paired.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_pilot_reg_012h_R1.fastq.gz.paired.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_pilot_stage_H2_000h_R1.fastq.gz.paired.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_pilot_reg_024h_R1.fastq.gz.paired.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_pilot_stage_H1_000h_R1.fastq.gz.paired.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_pilot_stage_H2long_000h_R1.fastq.gz.paired.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_pilot_reg_000h_R1.fastq.gz.paired.trimmed.fastq.gz\"]\n",
        "pairB = [\"/mlig_raw/trim/mlig_pilot_reg_096h_R2.fastq.gz.paired.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_pilot_reg_072h_R2.fastq.gz.paired.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_pilot_reg_048h_R2.fastq.gz.paired.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_pilot_reg_012h_R2.fastq.gz.paired.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_pilot_stage_H2_000h_R2.fastq.gz.paired.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_pilot_reg_024h_R2.fastq.gz.paired.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_pilot_stage_H1_000h_R2.fastq.gz.paired.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_pilot_stage_H2long_000h_R2.fastq.gz.paired.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_pilot_reg_000h_R2.fastq.gz.paired.trimmed.fastq.gz\"]\n",
        "\n",
        "outA = open(\"left.stranded.fastq\",'w')\n",
        "outB = open(\"right.stranded.fastq\",'w')\n",
        "\n",
        "aCount = 0\n",
        "for a in pairA:\n",
        "    name = a.split('/')[-1].split('.')[0][:-3]\n",
        "\n",
        "    stream = gzip.open(a,'rb')\n",
        "    for line in stream:\n",
        "        seq = stream.next()\n",
        "        stream.next()\n",
        "        qual = stream.next()\n",
        "        outA.write('@r' + str(aCount).zfill(10) + '_' + name + '/1\\n' + seq + '+\\n' + qual)\n",
        "        aCount += 1\n",
        "\n",
        "bCount = 0\n",
        "for b in pairB:\n",
        "    name = b.split('/')[-1].split('.')[0][:-3]\n",
        "\n",
        "    stream = gzip.open(b,'rb')\n",
        "    for line in stream:\n",
        "        seq = stream.next()\n",
        "        stream.next()\n",
        "        qual = stream.next()\n",
        "        outB.write('@r' + str(bCount).zfill(10) + '_' + name + '/2\\n' + seq + '+\\n' + qual)\n",
        "        bCount += 1\n",
        "\n",
        "outA.close()\n",
        "outB.close()\n",
        "</textarea>"
       ],
       "metadata": {},
       "output_type": "pyout",
       "prompt_number": 3,
       "text": [
        "<IPython.core.display.HTML at 0x21cf1d0>"
       ]
      }
     ],
     "prompt_number": 3
    },
    {
     "cell_type": "code",
     "collapsed": false,
     "input": [
      "import json\n",
      "libs = []\n",
      "trimmLog = open('resource/trimSE.err')\n",
      "for line in trimmLog:\n",
      "    if line.find('Started with arguments') != -1:\n",
      "        libs.append(prefix + line.strip().split('.fastq.gz')[0].split()[-1].split('/')[-1] + '.trimmed.fastq.gz')\n",
      "trimmLog.close()\n",
      "\n",
      "HTML(textarea('Script for all SE reads','''import gzip\n",
      "allFiles = ''' + json.dumps(libs) + '''\n",
      "\n",
      "outA = open(\"all.se.fastq\",'w')\n",
      "\n",
      "aCount = 0\n",
      "for a in allFiles:\n",
      "    name = a.split('/')[-1].split('.')[0][:-3]\n",
      "\n",
      "    stream = gzip.open(a,'rb')\n",
      "    for line in stream:\n",
      "        seq = stream.next()\n",
      "        stream.next()\n",
      "        qual = stream.next()\n",
      "        outA.write('@r' + str(aCount).zfill(10) + '_' + name + '\\\\n' + seq + '+\\\\n' + qual)\n",
      "        aCount += 1\n",
      "\n",
      "outA.close()'''))"
     ],
     "language": "python",
     "metadata": {},
     "outputs": [
      {
       "html": [
        "<b>Script for all SE reads</b>:<br>\n",
        "<textarea style=\"width: 1000px; height:200px\">\n",
        "import gzip\n",
        "allFiles = [\"/mlig_raw/trim/mlig_regen_reg_096h_3_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_reg_168h_2_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_reg_072h_3_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_control_072h_3_2.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_reg_012h_3_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_control_072h_2_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_reg_012h_1_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_reg_120h_1_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_reg_096h_3_2.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_control_168h_2_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_control_168h_1_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_control_096h_3_2.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_control_096h_2_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_reg_000h_1_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_reg_012h_2_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_control_012h_1_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_control_048h_3_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_control_012h_2_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_reg_024h_1_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_reg_012h_3_2.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_control_024h_3_2.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_reg_120h_3_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_reg_000h_3_2.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_reg_048h_3_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_control_012h_3_2.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_control_012h_3_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_control_000h_3_2.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_reg_048h_2_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_control_024h_3_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_reg_096h_1_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_control_024h_2_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_reg_168h_1_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_reg_120h_2_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_control_120h_1_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_reg_072h_3_2.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_reg_072h_1_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_reg_000h_3_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_reg_024h_3_2.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_control_096h_3_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_control_024h_1_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_reg_072h_2_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_reg_000h_2_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_control_000h_3_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_reg_168h_3_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_control_072h_1_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_reg_048h_1_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_reg_048h_3_2.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_reg_096h_2_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_reg_024h_3_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_control_096h_1_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_control_120h_3_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_control_168h_3_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_reg_024h_2_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_control_048h_1_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_control_120h_2_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_control_072h_3_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_control_048h_2_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_control_048h_3_2.trimmed.fastq.gz\"]\n",
        "\n",
        "outA = open(\"all.se.fastq\",'w')\n",
        "\n",
        "aCount = 0\n",
        "for a in allFiles:\n",
        "    name = a.split('/')[-1].split('.')[0][:-3]\n",
        "\n",
        "    stream = gzip.open(a,'rb')\n",
        "    for line in stream:\n",
        "        seq = stream.next()\n",
        "        stream.next()\n",
        "        qual = stream.next()\n",
        "        outA.write('@r' + str(aCount).zfill(10) + '_' + name + '\\n' + seq + '+\\n' + qual)\n",
        "        aCount += 1\n",
        "\n",
        "outA.close()\n",
        "</textarea>"
       ],
       "metadata": {},
       "output_type": "pyout",
       "prompt_number": 20,
       "text": [
        "<IPython.core.display.HTML at 0x1eef390>"
       ]
      }
     ],
     "prompt_number": 20
    },
    {
     "cell_type": "code",
     "collapsed": false,
     "input": [
      "import json\n",
      "trimmLog = open('resource/trimPE.err')\n",
      "libs = []\n",
      "prefix = '/mlig_raw/trim/'\n",
      "for line in trimmLog:\n",
      "    if line.find('Started with arguments') != -1:\n",
      "        libs.append(prefix + line.strip().split('.fastq.gz')[0].split()[-1].split('/')[-1][:-1] + '1.fastq.gz.paired.trimmed.fastq.gz')\n",
      "        libs.append(prefix + line.strip().split('.fastq.gz')[0].split()[-1].split('/')[-1][:-1] + '2.fastq.gz.paired.trimmed.fastq.gz')\n",
      "        libs.append(prefix + line.strip().split('.fastq.gz')[0].split()[-1].split('/')[-1][:-1] + '1.fastq.gz.unpaired.trimmed.fastq.gz')\n",
      "        libs.append(prefix + line.strip().split('.fastq.gz')[0].split()[-1].split('/')[-1][:-1] + '2.fastq.gz.unpaired.trimmed.fastq.gz')\n",
      "trimmLog.close()\n",
      "trimmLog = open('resource/trimSE.err')\n",
      "for line in trimmLog:\n",
      "    if line.find('Started with arguments') != -1:\n",
      "        libs.append(prefix + line.strip().split('.fastq.gz')[0].split()[-1].split('/')[-1] + '.trimmed.fastq.gz')\n",
      "trimmLog.close()\n",
      "\n",
      "HTML(textarea('Script for consolidating all libraries as single reads for inchworm k-mer analysis','''import gzip\n",
      "allFiles = ''' + json.dumps(libs) + '''\n",
      "\n",
      "outA = open(\"all.fasta\",'w')\n",
      "\n",
      "aCount = 0\n",
      "for a in allFiles:\n",
      "    name = a.split('/')[-1].split('.')[0][:-3]\n",
      "\n",
      "    stream = gzip.open(a,'rb')\n",
      "    for line in stream:\n",
      "        seq = stream.next()\n",
      "        stream.next()\n",
      "        qual = stream.next()\n",
      "        outA.write('>r' + str(aCount).zfill(10) + '_' + name + '\\\\n' + seq)\n",
      "        aCount += 1\n",
      "\n",
      "outA.close()'''))"
     ],
     "language": "python",
     "metadata": {},
     "outputs": [
      {
       "html": [
        "<b>Script for consolidating all libraries as single reads for inchworm k-mer analysis</b>:<br>\n",
        "<textarea style=\"width: 1000px; height:200px\">\n",
        "import gzip\n",
        "allFiles = [\"/mlig_raw/trim/mlig_pilot_reg_096h_R1.fastq.gz.paired.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_pilot_reg_096h_R2.fastq.gz.paired.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_pilot_reg_096h_R1.fastq.gz.unpaired.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_pilot_reg_096h_R2.fastq.gz.unpaired.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_pilot_reg_072h_R1.fastq.gz.paired.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_pilot_reg_072h_R2.fastq.gz.paired.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_pilot_reg_072h_R1.fastq.gz.unpaired.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_pilot_reg_072h_R2.fastq.gz.unpaired.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_pilot_reg_048h_R1.fastq.gz.paired.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_pilot_reg_048h_R2.fastq.gz.paired.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_pilot_reg_048h_R1.fastq.gz.unpaired.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_pilot_reg_048h_R2.fastq.gz.unpaired.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_pilot_stage_A1_000h_R1.fastq.gz.paired.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_pilot_stage_A1_000h_R2.fastq.gz.paired.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_pilot_stage_A1_000h_R1.fastq.gz.unpaired.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_pilot_stage_A1_000h_R2.fastq.gz.unpaired.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_pilot_reg_012h_R1.fastq.gz.paired.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_pilot_reg_012h_R2.fastq.gz.paired.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_pilot_reg_012h_R1.fastq.gz.unpaired.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_pilot_reg_012h_R2.fastq.gz.unpaired.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_pilot_stage_H2_000h_R1.fastq.gz.paired.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_pilot_stage_H2_000h_R2.fastq.gz.paired.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_pilot_stage_H2_000h_R1.fastq.gz.unpaired.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_pilot_stage_H2_000h_R2.fastq.gz.unpaired.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_pilot_reg_024h_R1.fastq.gz.paired.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_pilot_reg_024h_R2.fastq.gz.paired.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_pilot_reg_024h_R1.fastq.gz.unpaired.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_pilot_reg_024h_R2.fastq.gz.unpaired.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_pilot_stage_H1_000h_R1.fastq.gz.paired.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_pilot_stage_H1_000h_R2.fastq.gz.paired.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_pilot_stage_H1_000h_R1.fastq.gz.unpaired.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_pilot_stage_H1_000h_R2.fastq.gz.unpaired.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_pilot_stage_A2_000h_R1.fastq.gz.paired.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_pilot_stage_A2_000h_R2.fastq.gz.paired.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_pilot_stage_A2_000h_R1.fastq.gz.unpaired.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_pilot_stage_A2_000h_R2.fastq.gz.unpaired.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_pilot_stage_H2long_000h_R1.fastq.gz.paired.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_pilot_stage_H2long_000h_R2.fastq.gz.paired.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_pilot_stage_H2long_000h_R1.fastq.gz.unpaired.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_pilot_stage_H2long_000h_R2.fastq.gz.unpaired.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_pilot_reg_000h_R1.fastq.gz.paired.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_pilot_reg_000h_R2.fastq.gz.paired.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_pilot_reg_000h_R1.fastq.gz.unpaired.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_pilot_reg_000h_R2.fastq.gz.unpaired.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_reg_096h_3_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_reg_168h_2_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_reg_072h_3_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_control_072h_3_2.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_reg_012h_3_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_control_072h_2_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_reg_012h_1_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_reg_120h_1_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_reg_096h_3_2.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_control_168h_2_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_control_168h_1_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_control_096h_3_2.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_control_096h_2_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_reg_000h_1_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_reg_012h_2_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_control_012h_1_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_control_048h_3_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_control_012h_2_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_reg_024h_1_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_reg_012h_3_2.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_control_024h_3_2.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_reg_120h_3_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_reg_000h_3_2.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_reg_048h_3_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_control_012h_3_2.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_control_012h_3_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_control_000h_3_2.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_reg_048h_2_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_control_024h_3_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_reg_096h_1_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_control_024h_2_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_reg_168h_1_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_reg_120h_2_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_control_120h_1_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_reg_072h_3_2.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_reg_072h_1_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_reg_000h_3_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_reg_024h_3_2.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_control_096h_3_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_control_024h_1_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_reg_072h_2_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_reg_000h_2_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_control_000h_3_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_reg_168h_3_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_control_072h_1_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_reg_048h_1_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_reg_048h_3_2.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_reg_096h_2_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_reg_024h_3_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_control_096h_1_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_control_120h_3_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_control_168h_3_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_reg_024h_2_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_control_048h_1_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_control_120h_2_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_control_072h_3_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_control_048h_2_1.trimmed.fastq.gz\", \"/mlig_raw/trim/mlig_regen_control_048h_3_2.trimmed.fastq.gz\"]\n",
        "\n",
        "outA = open(\"all.fasta\",'w')\n",
        "\n",
        "aCount = 0\n",
        "for a in allFiles:\n",
        "    name = a.split('/')[-1].split('.')[0][:-3]\n",
        "\n",
        "    stream = gzip.open(a,'rb')\n",
        "    for line in stream:\n",
        "        seq = stream.next()\n",
        "        stream.next()\n",
        "        qual = stream.next()\n",
        "        outA.write('>r' + str(aCount).zfill(10) + '_' + name + '\\n' + seq)\n",
        "        aCount += 1\n",
        "\n",
        "outA.close()\n",
        "</textarea>"
       ],
       "metadata": {},
       "output_type": "pyout",
       "prompt_number": 21,
       "text": [
        "<IPython.core.display.HTML at 0x1efce90>"
       ]
      }
     ],
     "prompt_number": 21
    },
    {
     "cell_type": "code",
     "collapsed": false,
     "input": [
      "#Shuffle the raw sequence fasta file with this script. Not the best way to do this. Will still have some systematic bias.\n",
      "#Shuffled with 5 rounds\n",
      "HTML(textarea('Shuffle script','''import random, os, sys\n",
      "\n",
      "def reShuffle(filePath, outDir, cycles = 0):\n",
      "    binNum = random.randint(300,500)\n",
      "    faFile = open(filePath)\n",
      "\n",
      "    binFiles = {}\n",
      "    for i in range(binNum):\n",
      "        name = 'bin.' + str(i + 1).zfill(3) + '.fa'\n",
      "        binFiles[i + 1] = open(outDir + name,'w')\n",
      "\n",
      "    for line in faFile:\n",
      "        stream = binFiles[random.randint(1,binNum)]\n",
      "        stream.write(line)\n",
      "        stream.write(faFile.next())\n",
      "\n",
      "    faFile.close()\n",
      "    for stream in binFiles.values():\n",
      "        stream.close()\n",
      "\n",
      "    os.system('cat ' + outDir + 'bin*.fa > ' + outDir + 'shuffled.fa')\n",
      "    os.system('rm -f ' + outDir + 'bin*.fa')\n",
      "\n",
      "    if cycles == 0:\n",
      "        print 'Finished'\n",
      "    else:\n",
      "        print 'Finished, ' + str(cycles) + ' cycles remaining'\n",
      "        reShuffle(outDir + 'shuffled.fa', outDir, cycles - 1)\n",
      "    \n",
      "reShuffle(sys.argv[1],sys.argv[2],int(sys.argv[3]))'''))"
     ],
     "language": "python",
     "metadata": {},
     "outputs": [
      {
       "html": [
        "<b>Shuffle script</b>:<br>\n",
        "<textarea style=\"width: 1000px; height:200px\">\n",
        "import random, os, sys\n",
        "\n",
        "def reShuffle(filePath, outDir, cycles = 0):\n",
        "    binNum = random.randint(300,500)\n",
        "    faFile = open(filePath)\n",
        "\n",
        "    binFiles = {}\n",
        "    for i in range(binNum):\n",
        "        name = 'bin.' + str(i + 1).zfill(3) + '.fa'\n",
        "        binFiles[i + 1] = open(outDir + name,'w')\n",
        "\n",
        "    for line in faFile:\n",
        "        stream = binFiles[random.randint(1,binNum)]\n",
        "        stream.write(line)\n",
        "        stream.write(faFile.next())\n",
        "\n",
        "    faFile.close()\n",
        "    for stream in binFiles.values():\n",
        "        stream.close()\n",
        "\n",
        "    os.system('cat ' + outDir + 'bin*.fa > ' + outDir + 'shuffled.fa')\n",
        "    os.system('rm -f ' + outDir + 'bin*.fa')\n",
        "\n",
        "    if cycles == 0:\n",
        "        print 'Finished'\n",
        "    else:\n",
        "        print 'Finished, ' + str(cycles) + ' cycles remaining'\n",
        "        reShuffle(outDir + 'shuffled.fa', outDir, cycles - 1)\n",
        "    \n",
        "reShuffle(sys.argv[1],sys.argv[2],int(sys.argv[3]))\n",
        "</textarea>"
       ],
       "metadata": {},
       "output_type": "pyout",
       "prompt_number": 17,
       "text": [
        "<IPython.core.display.HTML at 0x1ef13d0>"
       ]
      }
     ],
     "prompt_number": 17
    },
    {
     "cell_type": "code",
     "collapsed": false,
     "input": [],
     "language": "python",
     "metadata": {},
     "outputs": []
    }
   ],
   "metadata": {}
  }
 ]
}