{
 "metadata": {
  "name": "",
  "signature": "sha256:8dd61d92c4cd15ae1a90260100a5017ae784c0820a23036f79717394c902f0a1"
 },
 "nbformat": 3,
 "nbformat_minor": 0,
 "worksheets": [
  {
   "cells": [
    {
     "cell_type": "markdown",
     "metadata": {},
     "source": [
      "# Consolidated assemblies\n",
      "\n",
      " 1. Run CD-HIT-EST on assemblies grouped by assembly method (oases, trinity, transabyss...)\n",
      "         cat trinity.*.fa > trinity.x.fa\n",
      "         cd-hit-est -i trinity.x.fa -o trinity.x.cdhit.fa -M 0 -T 32 -c 0.98 -d 0\n",
      "\n",
      " 2. Map reads uniquely back to CD-HIT-EST assemblies\n",
      " 3. Reads that map to multiple assembly methods are recorded\n",
      " 4. Cluster base on multi-method-mapped reads"
     ]
    },
    {
     "cell_type": "code",
     "collapsed": false,
     "input": [
      "import gzip, os, sys\n",
      "#import custom functions for displaying tables, bash commands\n",
      "sys.path.append(os.path.abspath(\"/home/damian/\"))\n",
      "from dk_ipython import *\n",
      "from IPython.display import HTML\n",
      "import json\n",
      "import pandas as pd\n",
      "import seaborn as sns\n",
      "import matplotlib.pyplot as plt\n",
      "\n",
      "%matplotlib inline\n",
      "HTML(addToggle())"
     ],
     "language": "python",
     "metadata": {},
     "outputs": [
      {
       "html": [
        "<script>\n",
        "code_show=true;\n",
        "function code_toggle() {\n",
        " if (code_show){\n",
        " $('div.input').hide();\n",
        " } else {\n",
        " $('div.input').show();\n",
        " }\n",
        " code_show = !code_show\n",
        "}\n",
        "\n",
        "$( document ).ready(code_toggle);\n",
        "</script>\n",
        "The raw code for this IPython notebook is by default hidden for easier reading.\n",
        "To toggle on/off the raw code, click <a href=\"javascript:code_toggle()\">here</a>."
       ],
       "metadata": {},
       "output_type": "pyout",
       "prompt_number": 16,
       "text": [
        "<IPython.core.display.HTML at 0x432cc9d0>"
       ]
      }
     ],
     "prompt_number": 16
    },
    {
     "cell_type": "markdown",
     "metadata": {},
     "source": [
      "### CD-HIT-EST and map"
     ]
    },
    {
     "cell_type": "code",
     "collapsed": false,
     "input": [
      "fs = [x for x in os.listdir('/home/share/ngsData/mlig/') if x.split('.')[-1] == 'gz']\n",
      "template = 'bowtie2 --very-fast --no-unal -p 32 -x EBWT -U FASTQ | samtools view -bS - > '\n",
      "cmds = ''\n",
      "\n",
      "for f in fs:\n",
      "    f = '/mlig/trim/' + f + '.paired.trimmed.fastq.gz'\n",
      "    cmds += template.replace('EBWT','/mlig/bt/oases').replace('FASTQ',f) + f.split('/')[-1].split('.')[0] + '.oases.bam\\n'\n",
      "    cmds += template.replace('EBWT','/mlig/bt/trinity').replace('FASTQ',f) + f.split('/')[-1].split('.')[0] + '.trinity.bam\\n'\n",
      "    cmds += template.replace('EBWT','/mlig/bt/soap').replace('FASTQ',f) + f.split('/')[-1].split('.')[0] + '.soap.bam\\n'\n",
      "    cmds += template.replace('EBWT','/mlig/bt/reassembly').replace('FASTQ',f) + f.split('/')[-1].split('.')[0] + '.reassembly.bam\\n'\n",
      "    cmds += template.replace('EBWT','/mlig/bt/transabyss').replace('FASTQ',f) + f.split('/')[-1].split('.')[0] + '.transabyss.bam\\n'\n",
      "    \n",
      "HTML(textarea('Bowtie2 mapping commands',cmds))"
     ],
     "language": "python",
     "metadata": {},
     "outputs": [
      {
       "html": [
        "<b>Bowtie2 mapping commands</b>:<br>\n",
        "<textarea style=\"width: 1000px; height:200px\">\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_pilot_reg_072h_R2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_reg_072h_R2.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_pilot_reg_072h_R2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_reg_072h_R2.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_pilot_reg_072h_R2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_reg_072h_R2.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_pilot_reg_072h_R2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_reg_072h_R2.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_pilot_reg_072h_R2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_reg_072h_R2.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_regen_reg_012h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_012h_3_1.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_regen_reg_012h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_012h_3_1.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_regen_reg_012h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_012h_3_1.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_regen_reg_012h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_012h_3_1.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_regen_reg_012h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_012h_3_1.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_regen_reg_072h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_072h_3_1.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_regen_reg_072h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_072h_3_1.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_regen_reg_072h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_072h_3_1.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_regen_reg_072h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_072h_3_1.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_regen_reg_072h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_072h_3_1.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_pilot_reg_024h_R2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_reg_024h_R2.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_pilot_reg_024h_R2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_reg_024h_R2.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_pilot_reg_024h_R2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_reg_024h_R2.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_pilot_reg_024h_R2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_reg_024h_R2.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_pilot_reg_024h_R2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_reg_024h_R2.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_regen_control_072h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_072h_2_1.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_regen_control_072h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_072h_2_1.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_regen_control_072h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_072h_2_1.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_regen_control_072h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_072h_2_1.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_regen_control_072h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_072h_2_1.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_pilot_reg_096h_R1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_reg_096h_R1.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_pilot_reg_096h_R1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_reg_096h_R1.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_pilot_reg_096h_R1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_reg_096h_R1.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_pilot_reg_096h_R1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_reg_096h_R1.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_pilot_reg_096h_R1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_reg_096h_R1.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_regen_control_072h_3_2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_072h_3_2.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_regen_control_072h_3_2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_072h_3_2.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_regen_control_072h_3_2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_072h_3_2.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_regen_control_072h_3_2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_072h_3_2.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_regen_control_072h_3_2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_072h_3_2.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_regen_reg_120h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_120h_1_1.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_regen_reg_120h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_120h_1_1.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_regen_reg_120h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_120h_1_1.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_regen_reg_120h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_120h_1_1.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_regen_reg_120h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_120h_1_1.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_regen_reg_168h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_168h_2_1.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_regen_reg_168h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_168h_2_1.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_regen_reg_168h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_168h_2_1.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_regen_reg_168h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_168h_2_1.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_regen_reg_168h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_168h_2_1.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_regen_reg_096h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_096h_3_1.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_regen_reg_096h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_096h_3_1.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_regen_reg_096h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_096h_3_1.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_regen_reg_096h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_096h_3_1.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_regen_reg_096h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_096h_3_1.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_regen_reg_012h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_012h_1_1.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_regen_reg_012h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_012h_1_1.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_regen_reg_012h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_012h_1_1.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_regen_reg_012h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_012h_1_1.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_regen_reg_012h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_012h_1_1.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_regen_control_168h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_168h_1_1.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_regen_control_168h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_168h_1_1.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_regen_control_168h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_168h_1_1.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_regen_control_168h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_168h_1_1.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_regen_control_168h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_168h_1_1.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_regen_reg_096h_3_2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_096h_3_2.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_regen_reg_096h_3_2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_096h_3_2.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_regen_reg_096h_3_2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_096h_3_2.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_regen_reg_096h_3_2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_096h_3_2.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_regen_reg_096h_3_2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_096h_3_2.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_regen_control_168h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_168h_2_1.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_regen_control_168h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_168h_2_1.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_regen_control_168h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_168h_2_1.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_regen_control_168h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_168h_2_1.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_regen_control_168h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_168h_2_1.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_regen_control_096h_3_2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_096h_3_2.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_regen_control_096h_3_2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_096h_3_2.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_regen_control_096h_3_2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_096h_3_2.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_regen_control_096h_3_2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_096h_3_2.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_regen_control_096h_3_2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_096h_3_2.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_regen_reg_000h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_000h_1_1.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_regen_reg_000h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_000h_1_1.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_regen_reg_000h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_000h_1_1.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_regen_reg_000h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_000h_1_1.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_regen_reg_000h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_000h_1_1.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_regen_control_096h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_096h_2_1.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_regen_control_096h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_096h_2_1.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_regen_control_096h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_096h_2_1.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_regen_control_096h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_096h_2_1.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_regen_control_096h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_096h_2_1.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_pilot_reg_048h_R1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_reg_048h_R1.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_pilot_reg_048h_R1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_reg_048h_R1.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_pilot_reg_048h_R1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_reg_048h_R1.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_pilot_reg_048h_R1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_reg_048h_R1.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_pilot_reg_048h_R1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_reg_048h_R1.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_pilot_reg_072h_R1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_reg_072h_R1.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_pilot_reg_072h_R1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_reg_072h_R1.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_pilot_reg_072h_R1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_reg_072h_R1.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_pilot_reg_072h_R1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_reg_072h_R1.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_pilot_reg_072h_R1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_reg_072h_R1.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_regen_control_012h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_012h_1_1.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_regen_control_012h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_012h_1_1.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_regen_control_012h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_012h_1_1.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_regen_control_012h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_012h_1_1.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_regen_control_012h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_012h_1_1.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_regen_reg_012h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_012h_2_1.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_regen_reg_012h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_012h_2_1.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_regen_reg_012h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_012h_2_1.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_regen_reg_012h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_012h_2_1.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_regen_reg_012h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_012h_2_1.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_pilot_reg_012h_R2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_reg_012h_R2.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_pilot_reg_012h_R2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_reg_012h_R2.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_pilot_reg_012h_R2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_reg_012h_R2.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_pilot_reg_012h_R2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_reg_012h_R2.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_pilot_reg_012h_R2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_reg_012h_R2.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_pilot_stage_A1_000h_R2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_A1_000h_R2.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_pilot_stage_A1_000h_R2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_A1_000h_R2.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_pilot_stage_A1_000h_R2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_A1_000h_R2.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_pilot_stage_A1_000h_R2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_A1_000h_R2.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_pilot_stage_A1_000h_R2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_A1_000h_R2.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_pilot_stage_A2340_000h_R1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_A2340_000h_R1.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_pilot_stage_A2340_000h_R1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_A2340_000h_R1.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_pilot_stage_A2340_000h_R1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_A2340_000h_R1.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_pilot_stage_A2340_000h_R1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_A2340_000h_R1.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_pilot_stage_A2340_000h_R1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_A2340_000h_R1.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_regen_control_012h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_012h_2_1.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_regen_control_012h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_012h_2_1.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_regen_control_012h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_012h_2_1.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_regen_control_012h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_012h_2_1.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_regen_control_012h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_012h_2_1.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_pilot_stage_A1350_000h_R1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_A1350_000h_R1.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_pilot_stage_A1350_000h_R1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_A1350_000h_R1.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_pilot_stage_A1350_000h_R1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_A1350_000h_R1.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_pilot_stage_A1350_000h_R1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_A1350_000h_R1.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_pilot_stage_A1350_000h_R1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_A1350_000h_R1.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_regen_control_048h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_048h_3_1.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_regen_control_048h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_048h_3_1.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_regen_control_048h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_048h_3_1.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_regen_control_048h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_048h_3_1.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_regen_control_048h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_048h_3_1.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_regen_reg_012h_3_2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_012h_3_2.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_regen_reg_012h_3_2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_012h_3_2.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_regen_reg_012h_3_2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_012h_3_2.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_regen_reg_012h_3_2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_012h_3_2.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_regen_reg_012h_3_2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_012h_3_2.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_regen_reg_024h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_024h_1_1.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_regen_reg_024h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_024h_1_1.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_regen_reg_024h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_024h_1_1.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_regen_reg_024h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_024h_1_1.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_regen_reg_024h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_024h_1_1.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_regen_control_024h_3_2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_024h_3_2.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_regen_control_024h_3_2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_024h_3_2.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_regen_control_024h_3_2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_024h_3_2.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_regen_control_024h_3_2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_024h_3_2.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_regen_control_024h_3_2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_024h_3_2.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_regen_reg_120h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_120h_3_1.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_regen_reg_120h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_120h_3_1.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_regen_reg_120h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_120h_3_1.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_regen_reg_120h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_120h_3_1.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_regen_reg_120h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_120h_3_1.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_regen_reg_000h_3_2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_000h_3_2.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_regen_reg_000h_3_2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_000h_3_2.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_regen_reg_000h_3_2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_000h_3_2.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_regen_reg_000h_3_2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_000h_3_2.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_regen_reg_000h_3_2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_000h_3_2.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_pilot_stage_A1_000h_R1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_A1_000h_R1.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_pilot_stage_A1_000h_R1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_A1_000h_R1.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_pilot_stage_A1_000h_R1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_A1_000h_R1.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_pilot_stage_A1_000h_R1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_A1_000h_R1.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_pilot_stage_A1_000h_R1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_A1_000h_R1.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_regen_reg_048h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_048h_3_1.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_regen_reg_048h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_048h_3_1.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_regen_reg_048h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_048h_3_1.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_regen_reg_048h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_048h_3_1.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_regen_reg_048h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_048h_3_1.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_regen_control_012h_3_2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_012h_3_2.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_regen_control_012h_3_2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_012h_3_2.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_regen_control_012h_3_2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_012h_3_2.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_regen_control_012h_3_2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_012h_3_2.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_regen_control_012h_3_2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_012h_3_2.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_regen_control_012h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_012h_3_1.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_regen_control_012h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_012h_3_1.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_regen_control_012h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_012h_3_1.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_regen_control_012h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_012h_3_1.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_regen_control_012h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_012h_3_1.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_regen_control_000h_3_2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_000h_3_2.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_regen_control_000h_3_2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_000h_3_2.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_regen_control_000h_3_2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_000h_3_2.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_regen_control_000h_3_2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_000h_3_2.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_regen_control_000h_3_2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_000h_3_2.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_pilot_stage_H2long_000h_R1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_H2long_000h_R1.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_pilot_stage_H2long_000h_R1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_H2long_000h_R1.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_pilot_stage_H2long_000h_R1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_H2long_000h_R1.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_pilot_stage_H2long_000h_R1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_H2long_000h_R1.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_pilot_stage_H2long_000h_R1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_H2long_000h_R1.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_pilot_reg_012h_R1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_reg_012h_R1.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_pilot_reg_012h_R1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_reg_012h_R1.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_pilot_reg_012h_R1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_reg_012h_R1.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_pilot_reg_012h_R1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_reg_012h_R1.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_pilot_reg_012h_R1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_reg_012h_R1.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_pilot_stage_A2340_000h_R2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_A2340_000h_R2.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_pilot_stage_A2340_000h_R2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_A2340_000h_R2.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_pilot_stage_A2340_000h_R2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_A2340_000h_R2.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_pilot_stage_A2340_000h_R2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_A2340_000h_R2.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_pilot_stage_A2340_000h_R2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_A2340_000h_R2.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_pilot_stage_H1_000h_R2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_H1_000h_R2.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_pilot_stage_H1_000h_R2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_H1_000h_R2.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_pilot_stage_H1_000h_R2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_H1_000h_R2.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_pilot_stage_H1_000h_R2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_H1_000h_R2.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_pilot_stage_H1_000h_R2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_H1_000h_R2.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_regen_control_024h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_024h_3_1.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_regen_control_024h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_024h_3_1.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_regen_control_024h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_024h_3_1.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_regen_control_024h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_024h_3_1.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_regen_control_024h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_024h_3_1.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_pilot_reg_024h_R1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_reg_024h_R1.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_pilot_reg_024h_R1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_reg_024h_R1.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_pilot_reg_024h_R1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_reg_024h_R1.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_pilot_reg_024h_R1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_reg_024h_R1.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_pilot_reg_024h_R1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_reg_024h_R1.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_regen_reg_048h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_048h_2_1.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_regen_reg_048h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_048h_2_1.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_regen_reg_048h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_048h_2_1.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_regen_reg_048h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_048h_2_1.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_regen_reg_048h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_048h_2_1.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_regen_reg_096h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_096h_1_1.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_regen_reg_096h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_096h_1_1.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_regen_reg_096h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_096h_1_1.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_regen_reg_096h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_096h_1_1.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_regen_reg_096h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_096h_1_1.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_regen_control_024h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_024h_2_1.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_regen_control_024h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_024h_2_1.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_regen_control_024h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_024h_2_1.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_regen_control_024h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_024h_2_1.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_regen_control_024h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_024h_2_1.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_regen_reg_168h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_168h_1_1.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_regen_reg_168h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_168h_1_1.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_regen_reg_168h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_168h_1_1.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_regen_reg_168h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_168h_1_1.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_regen_reg_168h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_168h_1_1.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_regen_reg_120h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_120h_2_1.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_regen_reg_120h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_120h_2_1.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_regen_reg_120h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_120h_2_1.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_regen_reg_120h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_120h_2_1.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_regen_reg_120h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_120h_2_1.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_regen_control_120h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_120h_1_1.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_regen_control_120h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_120h_1_1.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_regen_control_120h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_120h_1_1.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_regen_control_120h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_120h_1_1.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_regen_control_120h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_120h_1_1.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_regen_reg_072h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_072h_1_1.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_regen_reg_072h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_072h_1_1.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_regen_reg_072h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_072h_1_1.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_regen_reg_072h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_072h_1_1.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_regen_reg_072h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_072h_1_1.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_pilot_stage_A1350_000h_R2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_A1350_000h_R2.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_pilot_stage_A1350_000h_R2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_A1350_000h_R2.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_pilot_stage_A1350_000h_R2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_A1350_000h_R2.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_pilot_stage_A1350_000h_R2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_A1350_000h_R2.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_pilot_stage_A1350_000h_R2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_A1350_000h_R2.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_regen_reg_072h_3_2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_072h_3_2.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_regen_reg_072h_3_2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_072h_3_2.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_regen_reg_072h_3_2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_072h_3_2.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_regen_reg_072h_3_2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_072h_3_2.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_regen_reg_072h_3_2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_072h_3_2.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_regen_reg_024h_3_2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_024h_3_2.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_regen_reg_024h_3_2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_024h_3_2.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_regen_reg_024h_3_2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_024h_3_2.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_regen_reg_024h_3_2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_024h_3_2.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_regen_reg_024h_3_2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_024h_3_2.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_regen_reg_000h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_000h_3_1.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_regen_reg_000h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_000h_3_1.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_regen_reg_000h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_000h_3_1.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_regen_reg_000h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_000h_3_1.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_regen_reg_000h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_000h_3_1.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_regen_control_024h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_024h_1_1.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_regen_control_024h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_024h_1_1.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_regen_control_024h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_024h_1_1.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_regen_control_024h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_024h_1_1.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_regen_control_024h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_024h_1_1.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_regen_control_096h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_096h_3_1.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_regen_control_096h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_096h_3_1.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_regen_control_096h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_096h_3_1.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_regen_control_096h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_096h_3_1.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_regen_control_096h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_096h_3_1.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_regen_reg_072h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_072h_2_1.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_regen_reg_072h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_072h_2_1.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_regen_reg_072h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_072h_2_1.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_regen_reg_072h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_072h_2_1.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_regen_reg_072h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_072h_2_1.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_regen_reg_000h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_000h_2_1.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_regen_reg_000h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_000h_2_1.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_regen_reg_000h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_000h_2_1.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_regen_reg_000h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_000h_2_1.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_regen_reg_000h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_000h_2_1.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_regen_control_000h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_000h_3_1.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_regen_control_000h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_000h_3_1.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_regen_control_000h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_000h_3_1.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_regen_control_000h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_000h_3_1.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_regen_control_000h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_000h_3_1.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_regen_reg_168h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_168h_3_1.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_regen_reg_168h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_168h_3_1.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_regen_reg_168h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_168h_3_1.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_regen_reg_168h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_168h_3_1.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_regen_reg_168h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_168h_3_1.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_pilot_reg_000h_R2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_reg_000h_R2.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_pilot_reg_000h_R2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_reg_000h_R2.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_pilot_reg_000h_R2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_reg_000h_R2.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_pilot_reg_000h_R2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_reg_000h_R2.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_pilot_reg_000h_R2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_reg_000h_R2.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_regen_control_072h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_072h_1_1.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_regen_control_072h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_072h_1_1.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_regen_control_072h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_072h_1_1.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_regen_control_072h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_072h_1_1.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_regen_control_072h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_072h_1_1.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_pilot_stage_H1_000h_R1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_H1_000h_R1.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_pilot_stage_H1_000h_R1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_H1_000h_R1.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_pilot_stage_H1_000h_R1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_H1_000h_R1.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_pilot_stage_H1_000h_R1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_H1_000h_R1.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_pilot_stage_H1_000h_R1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_H1_000h_R1.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_regen_reg_048h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_048h_1_1.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_regen_reg_048h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_048h_1_1.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_regen_reg_048h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_048h_1_1.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_regen_reg_048h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_048h_1_1.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_regen_reg_048h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_048h_1_1.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_pilot_reg_048h_R2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_reg_048h_R2.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_pilot_reg_048h_R2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_reg_048h_R2.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_pilot_reg_048h_R2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_reg_048h_R2.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_pilot_reg_048h_R2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_reg_048h_R2.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_pilot_reg_048h_R2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_reg_048h_R2.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_regen_reg_096h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_096h_2_1.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_regen_reg_096h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_096h_2_1.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_regen_reg_096h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_096h_2_1.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_regen_reg_096h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_096h_2_1.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_regen_reg_096h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_096h_2_1.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_pilot_reg_000h_R1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_reg_000h_R1.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_pilot_reg_000h_R1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_reg_000h_R1.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_pilot_reg_000h_R1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_reg_000h_R1.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_pilot_reg_000h_R1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_reg_000h_R1.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_pilot_reg_000h_R1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_reg_000h_R1.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_regen_reg_024h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_024h_3_1.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_regen_reg_024h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_024h_3_1.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_regen_reg_024h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_024h_3_1.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_regen_reg_024h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_024h_3_1.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_regen_reg_024h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_024h_3_1.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_pilot_stage_A2_000h_R1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_A2_000h_R1.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_pilot_stage_A2_000h_R1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_A2_000h_R1.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_pilot_stage_A2_000h_R1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_A2_000h_R1.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_pilot_stage_A2_000h_R1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_A2_000h_R1.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_pilot_stage_A2_000h_R1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_A2_000h_R1.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_regen_reg_048h_3_2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_048h_3_2.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_regen_reg_048h_3_2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_048h_3_2.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_regen_reg_048h_3_2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_048h_3_2.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_regen_reg_048h_3_2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_048h_3_2.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_regen_reg_048h_3_2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_048h_3_2.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_pilot_stage_H2long_000h_R2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_H2long_000h_R2.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_pilot_stage_H2long_000h_R2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_H2long_000h_R2.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_pilot_stage_H2long_000h_R2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_H2long_000h_R2.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_pilot_stage_H2long_000h_R2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_H2long_000h_R2.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_pilot_stage_H2long_000h_R2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_H2long_000h_R2.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_pilot_stage_H2_000h_R2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_H2_000h_R2.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_pilot_stage_H2_000h_R2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_H2_000h_R2.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_pilot_stage_H2_000h_R2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_H2_000h_R2.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_pilot_stage_H2_000h_R2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_H2_000h_R2.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_pilot_stage_H2_000h_R2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_H2_000h_R2.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_regen_control_096h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_096h_1_1.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_regen_control_096h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_096h_1_1.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_regen_control_096h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_096h_1_1.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_regen_control_096h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_096h_1_1.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_regen_control_096h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_096h_1_1.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_regen_control_120h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_120h_2_1.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_regen_control_120h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_120h_2_1.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_regen_control_120h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_120h_2_1.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_regen_control_120h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_120h_2_1.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_regen_control_120h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_120h_2_1.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_regen_control_120h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_120h_3_1.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_regen_control_120h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_120h_3_1.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_regen_control_120h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_120h_3_1.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_regen_control_120h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_120h_3_1.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_regen_control_120h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_120h_3_1.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_regen_reg_024h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_024h_2_1.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_regen_reg_024h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_024h_2_1.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_regen_reg_024h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_024h_2_1.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_regen_reg_024h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_024h_2_1.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_regen_reg_024h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_reg_024h_2_1.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_regen_control_168h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_168h_3_1.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_regen_control_168h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_168h_3_1.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_regen_control_168h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_168h_3_1.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_regen_control_168h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_168h_3_1.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_regen_control_168h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_168h_3_1.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_regen_control_048h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_048h_1_1.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_regen_control_048h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_048h_1_1.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_regen_control_048h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_048h_1_1.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_regen_control_048h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_048h_1_1.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_regen_control_048h_1_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_048h_1_1.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_regen_control_048h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_048h_2_1.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_regen_control_048h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_048h_2_1.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_regen_control_048h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_048h_2_1.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_regen_control_048h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_048h_2_1.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_regen_control_048h_2_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_048h_2_1.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_regen_control_072h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_072h_3_1.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_regen_control_072h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_072h_3_1.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_regen_control_072h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_072h_3_1.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_regen_control_072h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_072h_3_1.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_regen_control_072h_3_1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_072h_3_1.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_pilot_stage_A2_000h_R2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_A2_000h_R2.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_pilot_stage_A2_000h_R2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_A2_000h_R2.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_pilot_stage_A2_000h_R2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_A2_000h_R2.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_pilot_stage_A2_000h_R2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_A2_000h_R2.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_pilot_stage_A2_000h_R2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_A2_000h_R2.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_pilot_stage_H2_000h_R1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_H2_000h_R1.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_pilot_stage_H2_000h_R1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_H2_000h_R1.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_pilot_stage_H2_000h_R1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_H2_000h_R1.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_pilot_stage_H2_000h_R1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_H2_000h_R1.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_pilot_stage_H2_000h_R1.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_stage_H2_000h_R1.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_pilot_reg_096h_R2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_reg_096h_R2.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_pilot_reg_096h_R2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_reg_096h_R2.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_pilot_reg_096h_R2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_reg_096h_R2.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_pilot_reg_096h_R2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_reg_096h_R2.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_pilot_reg_096h_R2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_pilot_reg_096h_R2.transabyss.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/oases -U /mlig/trim/mlig_regen_control_048h_3_2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_048h_3_2.oases.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/trinity -U /mlig/trim/mlig_regen_control_048h_3_2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_048h_3_2.trinity.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/soap -U /mlig/trim/mlig_regen_control_048h_3_2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_048h_3_2.soap.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/reassembly -U /mlig/trim/mlig_regen_control_048h_3_2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_048h_3_2.reassembly.bam\n",
        "bowtie2 --very-fast --no-unal -p 32 -x /mlig/bt/transabyss -U /mlig/trim/mlig_regen_control_048h_3_2.fastq.gz.paired.trimmed.fastq.gz | samtools view -bS - > mlig_regen_control_048h_3_2.transabyss.bam\n",
        "\n",
        "</textarea>"
       ],
       "metadata": {},
       "output_type": "pyout",
       "prompt_number": 33,
       "text": [
        "<IPython.core.display.HTML at 0x4d0bd50>"
       ]
      }
     ],
     "prompt_number": 33
    },
    {
     "cell_type": "markdown",
     "metadata": {},
     "source": [
      "### Blast transcripts to complete proteomes\n",
      "\n",
      "Transcripts were made into a blast database. Complete proteomes were blasted to the transcripts:\n",
      "\n",
      "    tblastn -query /home/share/proteome/sman.proteome.fa -db assembly.transcripts.cdhit.filter.fa -evalue 1e-5 -outfmt '6 qseqid sseqid qlen slen pident length qframe qstart qend sstart send evalue' -num_threads 4 > sman.blast.out\n",
      "    tblastn -query /home/share/proteome/cele.proteome.fa -db assembly.transcripts.cdhit.filter.fa -evalue 1e-5 -outfmt '6 qseqid sseqid qlen slen pident length qframe qstart qend sstart send evalue' -num_threads 4 > cele.blast.out\n",
      "    tblastn -query /home/share/proteome/dmel.proteome.fa -db assembly.transcripts.cdhit.filter.fa -evalue 1e-5 -outfmt '6 qseqid sseqid qlen slen pident length qframe qstart qend sstart send evalue' -num_threads 4 > dmel.blast.out\n",
      "    tblastn -query /home/share/proteome/hsap.proteome.fa -db assembly.transcripts.cdhit.filter.fa -evalue 1e-5 -outfmt '6 qseqid sseqid qlen slen pident length qframe qstart qend sstart send evalue' -num_threads 4 > hsap.blast.out\n",
      "    tblastn -query /home/share/proteome/csin.proteome.fa -db assembly.transcripts.cdhit.filter.fa -evalue 1e-5 -outfmt '6 qseqid sseqid qlen slen pident length qframe qstart qend sstart send evalue' -num_threads 4 > csin.blast.out\n",
      "    tblastn -query /home/share/proteome/emul.proteome.fa -db assembly.transcripts.cdhit.filter.fa -evalue 1e-5 -outfmt '6 qseqid sseqid qlen slen pident length qframe qstart qend sstart send evalue' -num_threads 4 > emul.blast.out\n",
      "    tblastx -query /home/share/transcriptome/smed/ox_smed_v1/ox_Smed_v1.fa -db assembly.transcripts.cdhit.filter.fa -evalue 1e-5 -outfmt '6 qseqid sseqid qlen slen pident length qframe qstart qend sstart send evalue' -num_threads 4 > smed.blast.out\n"
     ]
    },
    {
     "cell_type": "markdown",
     "metadata": {},
     "source": [
      "### Reformat locus/components"
     ]
    },
    {
     "cell_type": "code",
     "collapsed": false,
     "input": [
      "\n",
      "\n",
      "HTML(textarea('Script','''import sys\n",
      "f = open(sys.argv[1])\n",
      "\n",
      "from collections import defaultdict\n",
      "clusters = defaultdict(list)\n",
      "for line in f:\n",
      "    cols = line.strip().split()\n",
      "    meta = cols[0].split('.')\n",
      "\n",
      "    clusterID = ''\n",
      "    if meta[0] == 'oases':\n",
      "        clusterID = 'Locus_' + cols[1].split('_')[1]\n",
      "    elif meta[0] == 'trinity':\n",
      "        clusterID = 'c' + cols[1].split('_')[0][1:]\n",
      "    elif meta[0] == 'reassembly':\n",
      "        clusterID = 'c' + cols[1].split('_')[0]\n",
      "    elif meta[0] == 'soap':\n",
      "        if cols[1][0] == 'C':\n",
      "            clusterID = cols[1]\n",
      "        else:\n",
      "            clusterID = 'Locus_' + cols[2].split('_')[1]\n",
      "    elif meta[0] == 'transabyss':\n",
      "        clusterID = cols[1]\n",
      "    \n",
      "    c = '.'.join(cols[0].split('.')[:-1]) + '.' + clusterID\n",
      "    clusters[c].append(cols[0])\n",
      "\n",
      "for k,v in clusters.items():\n",
      "    print k + \"\\\\t\" + str(len(v)) + '\\\\t' + ','.join(v)\n",
      "'''))"
     ],
     "language": "python",
     "metadata": {},
     "outputs": [
      {
       "html": [
        "<b>Script</b>:<br>\n",
        "<textarea style=\"width: 1000px; height:200px\">\n",
        "import sys\n",
        "f = open(sys.argv[1])\n",
        "\n",
        "from collections import defaultdict\n",
        "clusters = defaultdict(list)\n",
        "for line in f:\n",
        "    cols = line.strip().split()\n",
        "    meta = cols[0].split('.')\n",
        "\n",
        "    clusterID = ''\n",
        "    if meta[0] == 'oases':\n",
        "        clusterID = 'Locus_' + cols[1].split('_')[1]\n",
        "    elif meta[0] == 'trinity':\n",
        "        clusterID = 'c' + cols[1].split('_')[0][1:]\n",
        "    elif meta[0] == 'reassembly':\n",
        "        clusterID = 'c' + cols[1].split('_')[0]\n",
        "    elif meta[0] == 'soap':\n",
        "        if cols[1][0] == 'C':\n",
        "            clusterID = cols[1]\n",
        "        else:\n",
        "            clusterID = 'Locus_' + cols[2].split('_')[1]\n",
        "    elif meta[0] == 'transabyss':\n",
        "        clusterID = cols[1]\n",
        "    \n",
        "    c = '.'.join(cols[0].split('.')[:-1]) + '.' + clusterID\n",
        "    clusters[c].append(cols[0])\n",
        "\n",
        "for k,v in clusters.items():\n",
        "    print k + \"\\t\" + str(len(v)) + '\\t' + ','.join(v)\n",
        "\n",
        "</textarea>"
       ],
       "metadata": {},
       "output_type": "pyout",
       "prompt_number": 62,
       "text": [
        "<IPython.core.display.HTML at 0x4d74150>"
       ]
      }
     ],
     "prompt_number": 62
    },
    {
     "cell_type": "markdown",
     "metadata": {},
     "source": [
      "### Extract unique mappings from .bam file"
     ]
    },
    {
     "cell_type": "code",
     "collapsed": false,
     "input": [
      "import pysam\n",
      "from collections import defaultdict\n",
      "\n",
      "prefix = '/home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/'\n",
      "fs = [x for x in os.listdir(prefix) if x.split('.')[-1] == 'bam']\n",
      "cmds = ''\n",
      "for f in fs:\n",
      "    cmds += 'samtools view -q 42 ' + prefix + f + ' | cut -f 1,3 > ' + '.'.join(f.split('.')[:-1]) + '.data\\n'\n",
      "    \n",
      "HTML(textarea('',cmds))"
     ],
     "language": "python",
     "metadata": {},
     "outputs": [
      {
       "html": [
        "<b></b>:<br>\n",
        "<textarea style=\"width: 1000px; height:200px\">\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_168h_3_1.soap.bam | cut -f 1,3 > mlig_regen_control_168h_3_1.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_A1350_000h_R2.oases.bam | cut -f 1,3 > mlig_pilot_stage_A1350_000h_R2.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_072h_1_1.oases.bam | cut -f 1,3 > mlig_regen_reg_072h_1_1.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_reg_072h_R1.oases.bam | cut -f 1,3 > mlig_pilot_reg_072h_R1.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_048h_2_1.transabyss.bam | cut -f 1,3 > mlig_regen_reg_048h_2_1.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_096h_3_1.soap.bam | cut -f 1,3 > mlig_regen_reg_096h_3_1.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_072h_3_1.oases.bam | cut -f 1,3 > mlig_regen_reg_072h_3_1.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_096h_2_1.soap.bam | cut -f 1,3 > mlig_regen_reg_096h_2_1.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_reg_096h_R1.transabyss.bam | cut -f 1,3 > mlig_pilot_reg_096h_R1.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_A2_000h_R2.transabyss.bam | cut -f 1,3 > mlig_pilot_stage_A2_000h_R2.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_120h_1_1.trinity.bam | cut -f 1,3 > mlig_regen_control_120h_1_1.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_000h_3_1.trinity.bam | cut -f 1,3 > mlig_regen_control_000h_3_1.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_A2340_000h_R1.oases.bam | cut -f 1,3 > mlig_pilot_stage_A2340_000h_R1.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_012h_3_2.oases.bam | cut -f 1,3 > mlig_regen_control_012h_3_2.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_A1350_000h_R1.reassembly.bam | cut -f 1,3 > mlig_pilot_stage_A1350_000h_R1.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_H2long_000h_R1.soap.bam | cut -f 1,3 > mlig_pilot_stage_H2long_000h_R1.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_072h_2_1.trinity.bam | cut -f 1,3 > mlig_regen_reg_072h_2_1.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_168h_1_1.oases.bam | cut -f 1,3 > mlig_regen_reg_168h_1_1.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_168h_1_1.reassembly.bam | cut -f 1,3 > mlig_regen_reg_168h_1_1.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_000h_3_1.reassembly.bam | cut -f 1,3 > mlig_regen_reg_000h_3_1.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_072h_1_1.trinity.bam | cut -f 1,3 > mlig_regen_control_072h_1_1.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_000h_3_1.soap.bam | cut -f 1,3 > mlig_regen_control_000h_3_1.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_reg_048h_R1.soap.bam | cut -f 1,3 > mlig_pilot_reg_048h_R1.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_072h_1_1.soap.bam | cut -f 1,3 > mlig_regen_reg_072h_1_1.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_072h_3_2.trinity.bam | cut -f 1,3 > mlig_regen_reg_072h_3_2.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_048h_3_1.transabyss.bam | cut -f 1,3 > mlig_regen_control_048h_3_1.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_168h_2_1.oases.bam | cut -f 1,3 > mlig_regen_reg_168h_2_1.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_048h_3_2.oases.bam | cut -f 1,3 > mlig_regen_reg_048h_3_2.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_reg_012h_R2.transabyss.bam | cut -f 1,3 > mlig_pilot_reg_012h_R2.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_reg_012h_R2.oases.bam | cut -f 1,3 > mlig_pilot_reg_012h_R2.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_120h_1_1.soap.bam | cut -f 1,3 > mlig_regen_reg_120h_1_1.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_reg_012h_R1.trinity.bam | cut -f 1,3 > mlig_pilot_reg_012h_R1.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_reg_012h_R2.soap.bam | cut -f 1,3 > mlig_pilot_reg_012h_R2.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_A1_000h_R1.transabyss.bam | cut -f 1,3 > mlig_pilot_stage_A1_000h_R1.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_reg_048h_R1.reassembly.bam | cut -f 1,3 > mlig_pilot_reg_048h_R1.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_096h_3_1.reassembly.bam | cut -f 1,3 > mlig_regen_reg_096h_3_1.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_072h_2_1.trinity.bam | cut -f 1,3 > mlig_regen_control_072h_2_1.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_120h_3_1.transabyss.bam | cut -f 1,3 > mlig_regen_control_120h_3_1.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_072h_2_1.reassembly.bam | cut -f 1,3 > mlig_regen_reg_072h_2_1.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_072h_3_1.trinity.bam | cut -f 1,3 > mlig_regen_control_072h_3_1.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_A1_000h_R1.soap.bam | cut -f 1,3 > mlig_pilot_stage_A1_000h_R1.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_012h_3_1.soap.bam | cut -f 1,3 > mlig_regen_reg_012h_3_1.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_072h_3_1.trinity.bam | cut -f 1,3 > mlig_regen_reg_072h_3_1.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_024h_3_2.reassembly.bam | cut -f 1,3 > mlig_regen_control_024h_3_2.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_072h_3_2.transabyss.bam | cut -f 1,3 > mlig_regen_control_072h_3_2.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_072h_3_1.soap.bam | cut -f 1,3 > mlig_regen_control_072h_3_1.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_096h_2_1.transabyss.bam | cut -f 1,3 > mlig_regen_reg_096h_2_1.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_reg_024h_R1.trinity.bam | cut -f 1,3 > mlig_pilot_reg_024h_R1.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_reg_000h_R2.transabyss.bam | cut -f 1,3 > mlig_pilot_reg_000h_R2.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_reg_048h_R2.trinity.bam | cut -f 1,3 > mlig_pilot_reg_048h_R2.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_048h_3_1.trinity.bam | cut -f 1,3 > mlig_regen_control_048h_3_1.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_reg_072h_R2.reassembly.bam | cut -f 1,3 > mlig_pilot_reg_072h_R2.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_reg_048h_R2.oases.bam | cut -f 1,3 > mlig_pilot_reg_048h_R2.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_168h_1_1.transabyss.bam | cut -f 1,3 > mlig_regen_reg_168h_1_1.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_reg_048h_R2.soap.bam | cut -f 1,3 > mlig_pilot_reg_048h_R2.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_024h_3_1.transabyss.bam | cut -f 1,3 > mlig_regen_control_024h_3_1.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_096h_3_2.transabyss.bam | cut -f 1,3 > mlig_regen_control_096h_3_2.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_120h_3_1.oases.bam | cut -f 1,3 > mlig_regen_reg_120h_3_1.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_reg_024h_R2.oases.bam | cut -f 1,3 > mlig_pilot_reg_024h_R2.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_012h_2_1.trinity.bam | cut -f 1,3 > mlig_regen_reg_012h_2_1.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_096h_3_1.oases.bam | cut -f 1,3 > mlig_regen_reg_096h_3_1.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_024h_3_1.reassembly.bam | cut -f 1,3 > mlig_regen_reg_024h_3_1.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_reg_096h_R2.trinity.bam | cut -f 1,3 > mlig_pilot_reg_096h_R2.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_A2_000h_R2.reassembly.bam | cut -f 1,3 > mlig_pilot_stage_A2_000h_R2.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_reg_096h_R1.oases.bam | cut -f 1,3 > mlig_pilot_reg_096h_R1.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_A2_000h_R1.reassembly.bam | cut -f 1,3 > mlig_pilot_stage_A2_000h_R1.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_048h_3_2.reassembly.bam | cut -f 1,3 > mlig_regen_reg_048h_3_2.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_H2long_000h_R2.reassembly.bam | cut -f 1,3 > mlig_pilot_stage_H2long_000h_R2.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_A1_000h_R2.oases.bam | cut -f 1,3 > mlig_pilot_stage_A1_000h_R2.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_168h_1_1.soap.bam | cut -f 1,3 > mlig_regen_reg_168h_1_1.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_H2_000h_R2.transabyss.bam | cut -f 1,3 > mlig_pilot_stage_H2_000h_R2.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_024h_3_2.soap.bam | cut -f 1,3 > mlig_regen_reg_024h_3_2.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_096h_1_1.soap.bam | cut -f 1,3 > mlig_regen_control_096h_1_1.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_024h_3_1.trinity.bam | cut -f 1,3 > mlig_regen_control_024h_3_1.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_012h_1_1.oases.bam | cut -f 1,3 > mlig_regen_control_012h_1_1.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_120h_2_1.reassembly.bam | cut -f 1,3 > mlig_regen_control_120h_2_1.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_048h_3_2.reassembly.bam | cut -f 1,3 > mlig_regen_control_048h_3_2.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_A2_000h_R1.soap.bam | cut -f 1,3 > mlig_pilot_stage_A2_000h_R1.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_096h_2_1.oases.bam | cut -f 1,3 > mlig_regen_control_096h_2_1.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_096h_3_1.oases.bam | cut -f 1,3 > mlig_regen_control_096h_3_1.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_H2long_000h_R1.trinity.bam | cut -f 1,3 > mlig_pilot_stage_H2long_000h_R1.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_048h_2_1.soap.bam | cut -f 1,3 > mlig_regen_reg_048h_2_1.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_reg_000h_R2.reassembly.bam | cut -f 1,3 > mlig_pilot_reg_000h_R2.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_120h_2_1.reassembly.bam | cut -f 1,3 > mlig_regen_reg_120h_2_1.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_H2long_000h_R1.transabyss.bam | cut -f 1,3 > mlig_pilot_stage_H2long_000h_R1.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_072h_2_1.oases.bam | cut -f 1,3 > mlig_regen_control_072h_2_1.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_096h_2_1.soap.bam | cut -f 1,3 > mlig_regen_control_096h_2_1.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_000h_2_1.soap.bam | cut -f 1,3 > mlig_regen_reg_000h_2_1.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_168h_2_1.soap.bam | cut -f 1,3 > mlig_regen_control_168h_2_1.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_072h_3_1.transabyss.bam | cut -f 1,3 > mlig_regen_control_072h_3_1.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_048h_3_2.trinity.bam | cut -f 1,3 > mlig_regen_control_048h_3_2.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_A2340_000h_R2.soap.bam | cut -f 1,3 > mlig_pilot_stage_A2340_000h_R2.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_120h_3_1.reassembly.bam | cut -f 1,3 > mlig_regen_control_120h_3_1.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_168h_3_1.trinity.bam | cut -f 1,3 > mlig_regen_reg_168h_3_1.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_reg_048h_R1.trinity.bam | cut -f 1,3 > mlig_pilot_reg_048h_R1.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_024h_3_2.soap.bam | cut -f 1,3 > mlig_regen_control_024h_3_2.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_024h_1_1.reassembly.bam | cut -f 1,3 > mlig_regen_control_024h_1_1.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_A1_000h_R2.transabyss.bam | cut -f 1,3 > mlig_pilot_stage_A1_000h_R2.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_072h_3_1.soap.bam | cut -f 1,3 > mlig_regen_reg_072h_3_1.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_012h_1_1.transabyss.bam | cut -f 1,3 > mlig_regen_reg_012h_1_1.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_A1350_000h_R1.soap.bam | cut -f 1,3 > mlig_pilot_stage_A1350_000h_R1.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_reg_012h_R1.transabyss.bam | cut -f 1,3 > mlig_pilot_reg_012h_R1.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_reg_048h_R1.oases.bam | cut -f 1,3 > mlig_pilot_reg_048h_R1.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_168h_2_1.oases.bam | cut -f 1,3 > mlig_regen_control_168h_2_1.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_000h_1_1.oases.bam | cut -f 1,3 > mlig_regen_reg_000h_1_1.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_024h_3_1.oases.bam | cut -f 1,3 > mlig_regen_reg_024h_3_1.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_reg_072h_R2.soap.bam | cut -f 1,3 > mlig_pilot_reg_072h_R2.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_H1_000h_R2.reassembly.bam | cut -f 1,3 > mlig_pilot_stage_H1_000h_R2.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_H2_000h_R2.soap.bam | cut -f 1,3 > mlig_pilot_stage_H2_000h_R2.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_A2_000h_R2.oases.bam | cut -f 1,3 > mlig_pilot_stage_A2_000h_R2.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_048h_1_1.oases.bam | cut -f 1,3 > mlig_regen_control_048h_1_1.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_A1_000h_R2.soap.bam | cut -f 1,3 > mlig_pilot_stage_A1_000h_R2.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_H2_000h_R1.soap.bam | cut -f 1,3 > mlig_pilot_stage_H2_000h_R1.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_012h_2_1.oases.bam | cut -f 1,3 > mlig_regen_reg_012h_2_1.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_000h_3_2.transabyss.bam | cut -f 1,3 > mlig_regen_reg_000h_3_2.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_048h_2_1.oases.bam | cut -f 1,3 > mlig_regen_control_048h_2_1.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_000h_2_1.transabyss.bam | cut -f 1,3 > mlig_regen_reg_000h_2_1.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_096h_3_1.reassembly.bam | cut -f 1,3 > mlig_regen_control_096h_3_1.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_A2_000h_R1.trinity.bam | cut -f 1,3 > mlig_pilot_stage_A2_000h_R1.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_012h_2_1.transabyss.bam | cut -f 1,3 > mlig_regen_reg_012h_2_1.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_024h_3_1.trinity.bam | cut -f 1,3 > mlig_regen_reg_024h_3_1.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_120h_3_1.oases.bam | cut -f 1,3 > mlig_regen_control_120h_3_1.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_024h_1_1.oases.bam | cut -f 1,3 > mlig_regen_reg_024h_1_1.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_072h_1_1.soap.bam | cut -f 1,3 > mlig_regen_control_072h_1_1.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_000h_2_1.trinity.bam | cut -f 1,3 > mlig_regen_reg_000h_2_1.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_096h_1_1.oases.bam | cut -f 1,3 > mlig_regen_control_096h_1_1.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_072h_3_2.transabyss.bam | cut -f 1,3 > mlig_regen_reg_072h_3_2.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_reg_024h_R1.oases.bam | cut -f 1,3 > mlig_pilot_reg_024h_R1.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_168h_2_1.trinity.bam | cut -f 1,3 > mlig_regen_control_168h_2_1.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_000h_3_1.transabyss.bam | cut -f 1,3 > mlig_regen_control_000h_3_1.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_168h_2_1.transabyss.bam | cut -f 1,3 > mlig_regen_control_168h_2_1.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_024h_2_1.soap.bam | cut -f 1,3 > mlig_regen_reg_024h_2_1.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_012h_2_1.soap.bam | cut -f 1,3 > mlig_regen_control_012h_2_1.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_072h_1_1.oases.bam | cut -f 1,3 > mlig_regen_control_072h_1_1.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_reg_024h_R1.reassembly.bam | cut -f 1,3 > mlig_pilot_reg_024h_R1.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_A2340_000h_R2.oases.bam | cut -f 1,3 > mlig_pilot_stage_A2340_000h_R2.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_024h_1_1.reassembly.bam | cut -f 1,3 > mlig_regen_reg_024h_1_1.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_024h_3_1.transabyss.bam | cut -f 1,3 > mlig_regen_reg_024h_3_1.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_048h_3_1.reassembly.bam | cut -f 1,3 > mlig_regen_reg_048h_3_1.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_reg_096h_R1.soap.bam | cut -f 1,3 > mlig_pilot_reg_096h_R1.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_072h_3_2.soap.bam | cut -f 1,3 > mlig_regen_reg_072h_3_2.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_000h_3_2.soap.bam | cut -f 1,3 > mlig_regen_reg_000h_3_2.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_A2340_000h_R1.soap.bam | cut -f 1,3 > mlig_pilot_stage_A2340_000h_R1.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_H1_000h_R2.soap.bam | cut -f 1,3 > mlig_pilot_stage_H1_000h_R2.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_120h_1_1.transabyss.bam | cut -f 1,3 > mlig_regen_reg_120h_1_1.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_048h_1_1.transabyss.bam | cut -f 1,3 > mlig_regen_control_048h_1_1.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_012h_1_1.oases.bam | cut -f 1,3 > mlig_regen_reg_012h_1_1.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_024h_3_2.oases.bam | cut -f 1,3 > mlig_regen_reg_024h_3_2.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_072h_1_1.reassembly.bam | cut -f 1,3 > mlig_regen_reg_072h_1_1.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_072h_3_2.soap.bam | cut -f 1,3 > mlig_regen_control_072h_3_2.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_012h_1_1.reassembly.bam | cut -f 1,3 > mlig_regen_control_012h_1_1.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_024h_3_2.reassembly.bam | cut -f 1,3 > mlig_regen_reg_024h_3_2.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_reg_012h_R1.reassembly.bam | cut -f 1,3 > mlig_pilot_reg_012h_R1.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_048h_1_1.trinity.bam | cut -f 1,3 > mlig_regen_control_048h_1_1.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_012h_3_2.oases.bam | cut -f 1,3 > mlig_regen_reg_012h_3_2.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_024h_1_1.soap.bam | cut -f 1,3 > mlig_regen_control_024h_1_1.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_000h_3_1.oases.bam | cut -f 1,3 > mlig_regen_reg_000h_3_1.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_048h_2_1.reassembly.bam | cut -f 1,3 > mlig_regen_control_048h_2_1.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_024h_3_2.trinity.bam | cut -f 1,3 > mlig_regen_reg_024h_3_2.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_168h_1_1.trinity.bam | cut -f 1,3 > mlig_regen_reg_168h_1_1.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_072h_2_1.oases.bam | cut -f 1,3 > mlig_regen_reg_072h_2_1.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_096h_1_1.reassembly.bam | cut -f 1,3 > mlig_regen_reg_096h_1_1.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_H2_000h_R1.reassembly.bam | cut -f 1,3 > mlig_pilot_stage_H2_000h_R1.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_168h_3_1.oases.bam | cut -f 1,3 > mlig_regen_reg_168h_3_1.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_012h_3_2.soap.bam | cut -f 1,3 > mlig_regen_control_012h_3_2.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_048h_2_1.reassembly.bam | cut -f 1,3 > mlig_regen_reg_048h_2_1.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_A2_000h_R2.soap.bam | cut -f 1,3 > mlig_pilot_stage_A2_000h_R2.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_012h_1_1.reassembly.bam | cut -f 1,3 > mlig_regen_reg_012h_1_1.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_012h_1_1.trinity.bam | cut -f 1,3 > mlig_regen_reg_012h_1_1.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_reg_096h_R2.soap.bam | cut -f 1,3 > mlig_pilot_reg_096h_R2.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_024h_1_1.transabyss.bam | cut -f 1,3 > mlig_regen_reg_024h_1_1.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_120h_2_1.trinity.bam | cut -f 1,3 > mlig_regen_reg_120h_2_1.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_reg_072h_R1.transabyss.bam | cut -f 1,3 > mlig_pilot_reg_072h_R1.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_reg_096h_R2.reassembly.bam | cut -f 1,3 > mlig_pilot_reg_096h_R2.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_reg_072h_R1.trinity.bam | cut -f 1,3 > mlig_pilot_reg_072h_R1.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_072h_2_1.soap.bam | cut -f 1,3 > mlig_regen_reg_072h_2_1.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_048h_1_1.transabyss.bam | cut -f 1,3 > mlig_regen_reg_048h_1_1.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_096h_3_1.transabyss.bam | cut -f 1,3 > mlig_regen_control_096h_3_1.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_012h_3_1.transabyss.bam | cut -f 1,3 > mlig_regen_reg_012h_3_1.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_048h_2_1.soap.bam | cut -f 1,3 > mlig_regen_control_048h_2_1.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_A1350_000h_R1.trinity.bam | cut -f 1,3 > mlig_pilot_stage_A1350_000h_R1.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_120h_1_1.oases.bam | cut -f 1,3 > mlig_regen_control_120h_1_1.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_048h_2_1.trinity.bam | cut -f 1,3 > mlig_regen_reg_048h_2_1.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_096h_2_1.reassembly.bam | cut -f 1,3 > mlig_regen_control_096h_2_1.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_120h_3_1.soap.bam | cut -f 1,3 > mlig_regen_reg_120h_3_1.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_024h_1_1.oases.bam | cut -f 1,3 > mlig_regen_control_024h_1_1.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_012h_3_1.transabyss.bam | cut -f 1,3 > mlig_regen_control_012h_3_1.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_A1350_000h_R1.oases.bam | cut -f 1,3 > mlig_pilot_stage_A1350_000h_R1.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_120h_1_1.transabyss.bam | cut -f 1,3 > mlig_regen_control_120h_1_1.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_072h_2_1.soap.bam | cut -f 1,3 > mlig_regen_control_072h_2_1.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_H2_000h_R2.reassembly.bam | cut -f 1,3 > mlig_pilot_stage_H2_000h_R2.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_H2_000h_R2.oases.bam | cut -f 1,3 > mlig_pilot_stage_H2_000h_R2.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_048h_3_2.trinity.bam | cut -f 1,3 > mlig_regen_reg_048h_3_2.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_048h_1_1.soap.bam | cut -f 1,3 > mlig_regen_control_048h_1_1.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_A2340_000h_R1.trinity.bam | cut -f 1,3 > mlig_pilot_stage_A2340_000h_R1.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_072h_3_1.transabyss.bam | cut -f 1,3 > mlig_regen_reg_072h_3_1.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_H2long_000h_R1.oases.bam | cut -f 1,3 > mlig_pilot_stage_H2long_000h_R1.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_012h_3_2.trinity.bam | cut -f 1,3 > mlig_regen_control_012h_3_2.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_072h_1_1.transabyss.bam | cut -f 1,3 > mlig_regen_reg_072h_1_1.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_120h_3_1.soap.bam | cut -f 1,3 > mlig_regen_control_120h_3_1.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_reg_012h_R1.oases.bam | cut -f 1,3 > mlig_pilot_reg_012h_R1.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_048h_3_1.oases.bam | cut -f 1,3 > mlig_regen_control_048h_3_1.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_000h_3_2.trinity.bam | cut -f 1,3 > mlig_regen_control_000h_3_2.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_012h_3_1.soap.bam | cut -f 1,3 > mlig_regen_control_012h_3_1.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_024h_2_1.trinity.bam | cut -f 1,3 > mlig_regen_reg_024h_2_1.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_168h_2_1.transabyss.bam | cut -f 1,3 > mlig_regen_reg_168h_2_1.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_012h_2_1.soap.bam | cut -f 1,3 > mlig_regen_reg_012h_2_1.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_120h_3_1.trinity.bam | cut -f 1,3 > mlig_regen_reg_120h_3_1.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_096h_3_2.trinity.bam | cut -f 1,3 > mlig_regen_control_096h_3_2.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_072h_1_1.reassembly.bam | cut -f 1,3 > mlig_regen_control_072h_1_1.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_168h_1_1.trinity.bam | cut -f 1,3 > mlig_regen_control_168h_1_1.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_048h_3_2.soap.bam | cut -f 1,3 > mlig_regen_reg_048h_3_2.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_H1_000h_R1.reassembly.bam | cut -f 1,3 > mlig_pilot_stage_H1_000h_R1.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_000h_1_1.trinity.bam | cut -f 1,3 > mlig_regen_reg_000h_1_1.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_reg_024h_R2.trinity.bam | cut -f 1,3 > mlig_pilot_reg_024h_R2.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_120h_1_1.reassembly.bam | cut -f 1,3 > mlig_regen_control_120h_1_1.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_012h_3_1.oases.bam | cut -f 1,3 > mlig_regen_reg_012h_3_1.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_072h_3_2.oases.bam | cut -f 1,3 > mlig_regen_reg_072h_3_2.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_A2340_000h_R2.reassembly.bam | cut -f 1,3 > mlig_pilot_stage_A2340_000h_R2.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_072h_3_1.oases.bam | cut -f 1,3 > mlig_regen_control_072h_3_1.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_reg_096h_R1.trinity.bam | cut -f 1,3 > mlig_pilot_reg_096h_R1.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_168h_3_1.oases.bam | cut -f 1,3 > mlig_regen_control_168h_3_1.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_012h_3_2.trinity.bam | cut -f 1,3 > mlig_regen_reg_012h_3_2.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_H1_000h_R1.transabyss.bam | cut -f 1,3 > mlig_pilot_stage_H1_000h_R1.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_000h_3_2.trinity.bam | cut -f 1,3 > mlig_regen_reg_000h_3_2.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_A2340_000h_R2.trinity.bam | cut -f 1,3 > mlig_pilot_stage_A2340_000h_R2.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_048h_1_1.soap.bam | cut -f 1,3 > mlig_regen_reg_048h_1_1.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_024h_3_1.reassembly.bam | cut -f 1,3 > mlig_regen_control_024h_3_1.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_120h_2_1.transabyss.bam | cut -f 1,3 > mlig_regen_control_120h_2_1.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_A2_000h_R1.transabyss.bam | cut -f 1,3 > mlig_pilot_stage_A2_000h_R1.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_H2_000h_R2.trinity.bam | cut -f 1,3 > mlig_pilot_stage_H2_000h_R2.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_000h_1_1.transabyss.bam | cut -f 1,3 > mlig_regen_reg_000h_1_1.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_096h_1_1.trinity.bam | cut -f 1,3 > mlig_regen_control_096h_1_1.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_012h_3_2.reassembly.bam | cut -f 1,3 > mlig_regen_reg_012h_3_2.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_168h_1_1.soap.bam | cut -f 1,3 > mlig_regen_control_168h_1_1.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_012h_3_1.trinity.bam | cut -f 1,3 > mlig_regen_reg_012h_3_1.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_012h_1_1.transabyss.bam | cut -f 1,3 > mlig_regen_control_012h_1_1.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_048h_3_1.soap.bam | cut -f 1,3 > mlig_regen_control_048h_3_1.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_048h_3_1.reassembly.bam | cut -f 1,3 > mlig_regen_control_048h_3_1.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_H1_000h_R2.transabyss.bam | cut -f 1,3 > mlig_pilot_stage_H1_000h_R2.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_000h_3_2.transabyss.bam | cut -f 1,3 > mlig_regen_control_000h_3_2.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_072h_3_1.reassembly.bam | cut -f 1,3 > mlig_regen_reg_072h_3_1.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_H1_000h_R1.oases.bam | cut -f 1,3 > mlig_pilot_stage_H1_000h_R1.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_reg_096h_R2.transabyss.bam | cut -f 1,3 > mlig_pilot_reg_096h_R2.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_A1350_000h_R2.trinity.bam | cut -f 1,3 > mlig_pilot_stage_A1350_000h_R2.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_048h_3_2.transabyss.bam | cut -f 1,3 > mlig_regen_reg_048h_3_2.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_000h_3_1.transabyss.bam | cut -f 1,3 > mlig_regen_reg_000h_3_1.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_048h_3_2.transabyss.bam | cut -f 1,3 > mlig_regen_control_048h_3_2.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_reg_048h_R1.transabyss.bam | cut -f 1,3 > mlig_pilot_reg_048h_R1.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_H2_000h_R1.oases.bam | cut -f 1,3 > mlig_pilot_stage_H2_000h_R1.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_reg_000h_R1.trinity.bam | cut -f 1,3 > mlig_pilot_reg_000h_R1.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_reg_048h_R2.transabyss.bam | cut -f 1,3 > mlig_pilot_reg_048h_R2.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_012h_3_1.reassembly.bam | cut -f 1,3 > mlig_regen_reg_012h_3_1.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_096h_1_1.oases.bam | cut -f 1,3 > mlig_regen_reg_096h_1_1.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_H1_000h_R1.soap.bam | cut -f 1,3 > mlig_pilot_stage_H1_000h_R1.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_024h_3_2.oases.bam | cut -f 1,3 > mlig_regen_control_024h_3_2.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_120h_3_1.transabyss.bam | cut -f 1,3 > mlig_regen_reg_120h_3_1.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_096h_1_1.soap.bam | cut -f 1,3 > mlig_regen_reg_096h_1_1.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_048h_3_1.soap.bam | cut -f 1,3 > mlig_regen_reg_048h_3_1.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_096h_3_2.soap.bam | cut -f 1,3 > mlig_regen_control_096h_3_2.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_024h_1_1.trinity.bam | cut -f 1,3 > mlig_regen_reg_024h_1_1.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_048h_1_1.oases.bam | cut -f 1,3 > mlig_regen_reg_048h_1_1.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_072h_3_2.oases.bam | cut -f 1,3 > mlig_regen_control_072h_3_2.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_096h_3_2.soap.bam | cut -f 1,3 > mlig_regen_reg_096h_3_2.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_reg_072h_R2.trinity.bam | cut -f 1,3 > mlig_pilot_reg_072h_R2.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_120h_1_1.reassembly.bam | cut -f 1,3 > mlig_regen_reg_120h_1_1.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_096h_3_2.trinity.bam | cut -f 1,3 > mlig_regen_reg_096h_3_2.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_072h_1_1.transabyss.bam | cut -f 1,3 > mlig_regen_control_072h_1_1.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_120h_2_1.soap.bam | cut -f 1,3 > mlig_regen_control_120h_2_1.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_120h_3_1.trinity.bam | cut -f 1,3 > mlig_regen_control_120h_3_1.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_168h_2_1.reassembly.bam | cut -f 1,3 > mlig_regen_control_168h_2_1.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_168h_2_1.soap.bam | cut -f 1,3 > mlig_regen_reg_168h_2_1.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_reg_000h_R1.oases.bam | cut -f 1,3 > mlig_pilot_reg_000h_R1.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_012h_3_2.transabyss.bam | cut -f 1,3 > mlig_regen_reg_012h_3_2.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_120h_1_1.soap.bam | cut -f 1,3 > mlig_regen_control_120h_1_1.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_reg_072h_R2.transabyss.bam | cut -f 1,3 > mlig_pilot_reg_072h_R2.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_096h_2_1.trinity.bam | cut -f 1,3 > mlig_regen_reg_096h_2_1.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_012h_1_1.soap.bam | cut -f 1,3 > mlig_regen_reg_012h_1_1.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_reg_000h_R2.trinity.bam | cut -f 1,3 > mlig_pilot_reg_000h_R2.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_096h_3_2.oases.bam | cut -f 1,3 > mlig_regen_control_096h_3_2.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_096h_1_1.reassembly.bam | cut -f 1,3 > mlig_regen_control_096h_1_1.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_168h_2_1.reassembly.bam | cut -f 1,3 > mlig_regen_reg_168h_2_1.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_H1_000h_R2.oases.bam | cut -f 1,3 > mlig_pilot_stage_H1_000h_R2.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_072h_3_2.reassembly.bam | cut -f 1,3 > mlig_regen_control_072h_3_2.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_reg_000h_R1.transabyss.bam | cut -f 1,3 > mlig_pilot_reg_000h_R1.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_168h_3_1.soap.bam | cut -f 1,3 > mlig_regen_reg_168h_3_1.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_048h_2_1.transabyss.bam | cut -f 1,3 > mlig_regen_control_048h_2_1.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_048h_1_1.trinity.bam | cut -f 1,3 > mlig_regen_reg_048h_1_1.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_000h_3_2.reassembly.bam | cut -f 1,3 > mlig_regen_control_000h_3_2.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_A2340_000h_R1.reassembly.bam | cut -f 1,3 > mlig_pilot_stage_A2340_000h_R1.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_048h_2_1.trinity.bam | cut -f 1,3 > mlig_regen_control_048h_2_1.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_024h_3_2.trinity.bam | cut -f 1,3 > mlig_regen_control_024h_3_2.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_reg_000h_R2.oases.bam | cut -f 1,3 > mlig_pilot_reg_000h_R2.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_000h_3_2.oases.bam | cut -f 1,3 > mlig_regen_control_000h_3_2.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_024h_2_1.oases.bam | cut -f 1,3 > mlig_regen_control_024h_2_1.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_120h_2_1.trinity.bam | cut -f 1,3 > mlig_regen_control_120h_2_1.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_096h_3_1.trinity.bam | cut -f 1,3 > mlig_regen_reg_096h_3_1.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_024h_2_1.transabyss.bam | cut -f 1,3 > mlig_regen_control_024h_2_1.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_048h_2_1.oases.bam | cut -f 1,3 > mlig_regen_reg_048h_2_1.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_048h_1_1.reassembly.bam | cut -f 1,3 > mlig_regen_reg_048h_1_1.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_A2_000h_R2.trinity.bam | cut -f 1,3 > mlig_pilot_stage_A2_000h_R2.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_168h_3_1.trinity.bam | cut -f 1,3 > mlig_regen_control_168h_3_1.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_096h_2_1.oases.bam | cut -f 1,3 > mlig_regen_reg_096h_2_1.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_096h_3_1.transabyss.bam | cut -f 1,3 > mlig_regen_reg_096h_3_1.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_072h_1_1.trinity.bam | cut -f 1,3 > mlig_regen_reg_072h_1_1.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_012h_2_1.reassembly.bam | cut -f 1,3 > mlig_regen_control_012h_2_1.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_120h_2_1.oases.bam | cut -f 1,3 > mlig_regen_reg_120h_2_1.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_048h_3_2.oases.bam | cut -f 1,3 > mlig_regen_control_048h_3_2.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_000h_2_1.reassembly.bam | cut -f 1,3 > mlig_regen_reg_000h_2_1.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_168h_3_1.reassembly.bam | cut -f 1,3 > mlig_regen_control_168h_3_1.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_120h_1_1.oases.bam | cut -f 1,3 > mlig_regen_reg_120h_1_1.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_reg_024h_R2.reassembly.bam | cut -f 1,3 > mlig_pilot_reg_024h_R2.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_120h_1_1.trinity.bam | cut -f 1,3 > mlig_regen_reg_120h_1_1.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_096h_3_2.transabyss.bam | cut -f 1,3 > mlig_regen_reg_096h_3_2.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_024h_3_2.transabyss.bam | cut -f 1,3 > mlig_regen_control_024h_3_2.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_A2340_000h_R2.transabyss.bam | cut -f 1,3 > mlig_pilot_stage_A2340_000h_R2.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_A2340_000h_R1.transabyss.bam | cut -f 1,3 > mlig_pilot_stage_A2340_000h_R1.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_096h_3_2.reassembly.bam | cut -f 1,3 > mlig_regen_reg_096h_3_2.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_048h_1_1.reassembly.bam | cut -f 1,3 > mlig_regen_control_048h_1_1.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_024h_2_1.trinity.bam | cut -f 1,3 > mlig_regen_control_024h_2_1.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_reg_000h_R1.reassembly.bam | cut -f 1,3 > mlig_pilot_reg_000h_R1.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_A1_000h_R2.reassembly.bam | cut -f 1,3 > mlig_pilot_stage_A1_000h_R2.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_A1_000h_R1.oases.bam | cut -f 1,3 > mlig_pilot_stage_A1_000h_R1.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_096h_1_1.transabyss.bam | cut -f 1,3 > mlig_regen_control_096h_1_1.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_168h_3_1.transabyss.bam | cut -f 1,3 > mlig_regen_control_168h_3_1.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_120h_2_1.transabyss.bam | cut -f 1,3 > mlig_regen_reg_120h_2_1.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_072h_3_2.reassembly.bam | cut -f 1,3 > mlig_regen_reg_072h_3_2.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_reg_012h_R2.reassembly.bam | cut -f 1,3 > mlig_pilot_reg_012h_R2.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_096h_3_2.oases.bam | cut -f 1,3 > mlig_regen_reg_096h_3_2.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_120h_3_1.reassembly.bam | cut -f 1,3 > mlig_regen_reg_120h_3_1.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_reg_024h_R1.transabyss.bam | cut -f 1,3 > mlig_pilot_reg_024h_R1.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_096h_3_2.reassembly.bam | cut -f 1,3 > mlig_regen_control_096h_3_2.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_A1_000h_R1.reassembly.bam | cut -f 1,3 > mlig_pilot_stage_A1_000h_R1.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_012h_3_2.soap.bam | cut -f 1,3 > mlig_regen_reg_012h_3_2.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_096h_3_1.soap.bam | cut -f 1,3 > mlig_regen_control_096h_3_1.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_H2long_000h_R2.trinity.bam | cut -f 1,3 > mlig_pilot_stage_H2long_000h_R2.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_reg_072h_R2.oases.bam | cut -f 1,3 > mlig_pilot_reg_072h_R2.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_072h_3_2.trinity.bam | cut -f 1,3 > mlig_regen_control_072h_3_2.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_A1350_000h_R1.transabyss.bam | cut -f 1,3 > mlig_pilot_stage_A1350_000h_R1.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_A1350_000h_R2.transabyss.bam | cut -f 1,3 > mlig_pilot_stage_A1350_000h_R2.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_reg_024h_R2.soap.bam | cut -f 1,3 > mlig_pilot_reg_024h_R2.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_012h_3_2.reassembly.bam | cut -f 1,3 > mlig_regen_control_012h_3_2.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_reg_072h_R1.reassembly.bam | cut -f 1,3 > mlig_pilot_reg_072h_R1.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_024h_1_1.soap.bam | cut -f 1,3 > mlig_regen_reg_024h_1_1.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_012h_3_1.oases.bam | cut -f 1,3 > mlig_regen_control_012h_3_1.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_168h_1_1.oases.bam | cut -f 1,3 > mlig_regen_control_168h_1_1.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_048h_3_1.trinity.bam | cut -f 1,3 > mlig_regen_reg_048h_3_1.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_H2_000h_R1.trinity.bam | cut -f 1,3 > mlig_pilot_stage_H2_000h_R1.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_072h_3_1.reassembly.bam | cut -f 1,3 > mlig_regen_control_072h_3_1.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_reg_048h_R2.reassembly.bam | cut -f 1,3 > mlig_pilot_reg_048h_R2.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_024h_2_1.oases.bam | cut -f 1,3 > mlig_regen_reg_024h_2_1.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_000h_1_1.reassembly.bam | cut -f 1,3 > mlig_regen_reg_000h_1_1.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_000h_1_1.soap.bam | cut -f 1,3 > mlig_regen_reg_000h_1_1.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_H2long_000h_R2.oases.bam | cut -f 1,3 > mlig_pilot_stage_H2long_000h_R2.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_024h_1_1.transabyss.bam | cut -f 1,3 > mlig_regen_control_024h_1_1.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_024h_2_1.soap.bam | cut -f 1,3 > mlig_regen_control_024h_2_1.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_048h_3_1.oases.bam | cut -f 1,3 > mlig_regen_reg_048h_3_1.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_012h_2_1.oases.bam | cut -f 1,3 > mlig_regen_control_012h_2_1.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_168h_1_1.transabyss.bam | cut -f 1,3 > mlig_regen_control_168h_1_1.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_reg_012h_R1.soap.bam | cut -f 1,3 > mlig_pilot_reg_012h_R1.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_reg_000h_R2.soap.bam | cut -f 1,3 > mlig_pilot_reg_000h_R2.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_012h_3_2.transabyss.bam | cut -f 1,3 > mlig_regen_control_012h_3_2.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_012h_2_1.transabyss.bam | cut -f 1,3 > mlig_regen_control_012h_2_1.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_120h_2_1.soap.bam | cut -f 1,3 > mlig_regen_reg_120h_2_1.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_168h_1_1.reassembly.bam | cut -f 1,3 > mlig_regen_control_168h_1_1.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_012h_2_1.trinity.bam | cut -f 1,3 > mlig_regen_control_012h_2_1.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_024h_3_1.soap.bam | cut -f 1,3 > mlig_regen_control_024h_3_1.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_096h_2_1.reassembly.bam | cut -f 1,3 > mlig_regen_reg_096h_2_1.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_H1_000h_R2.trinity.bam | cut -f 1,3 > mlig_pilot_stage_H1_000h_R2.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_A2_000h_R1.oases.bam | cut -f 1,3 > mlig_pilot_stage_A2_000h_R1.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_012h_3_1.trinity.bam | cut -f 1,3 > mlig_regen_control_012h_3_1.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_168h_3_1.transabyss.bam | cut -f 1,3 > mlig_regen_reg_168h_3_1.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_reg_096h_R2.oases.bam | cut -f 1,3 > mlig_pilot_reg_096h_R2.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_096h_1_1.transabyss.bam | cut -f 1,3 > mlig_regen_reg_096h_1_1.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_096h_2_1.trinity.bam | cut -f 1,3 > mlig_regen_control_096h_2_1.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_024h_2_1.transabyss.bam | cut -f 1,3 > mlig_regen_reg_024h_2_1.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_000h_3_1.reassembly.bam | cut -f 1,3 > mlig_regen_control_000h_3_1.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_H2_000h_R1.transabyss.bam | cut -f 1,3 > mlig_pilot_stage_H2_000h_R1.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_000h_3_1.trinity.bam | cut -f 1,3 > mlig_regen_reg_000h_3_1.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_H2long_000h_R1.reassembly.bam | cut -f 1,3 > mlig_pilot_stage_H2long_000h_R1.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_000h_3_1.oases.bam | cut -f 1,3 > mlig_regen_control_000h_3_1.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_reg_024h_R2.transabyss.bam | cut -f 1,3 > mlig_pilot_reg_024h_R2.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_000h_2_1.oases.bam | cut -f 1,3 > mlig_regen_reg_000h_2_1.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_012h_3_1.reassembly.bam | cut -f 1,3 > mlig_regen_control_012h_3_1.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_reg_000h_R1.soap.bam | cut -f 1,3 > mlig_pilot_reg_000h_R1.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_096h_3_1.trinity.bam | cut -f 1,3 > mlig_regen_control_096h_3_1.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_reg_096h_R1.reassembly.bam | cut -f 1,3 > mlig_pilot_reg_096h_R1.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_072h_2_1.reassembly.bam | cut -f 1,3 > mlig_regen_control_072h_2_1.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_012h_2_1.reassembly.bam | cut -f 1,3 > mlig_regen_reg_012h_2_1.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_024h_2_1.reassembly.bam | cut -f 1,3 > mlig_regen_control_024h_2_1.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_024h_3_2.transabyss.bam | cut -f 1,3 > mlig_regen_reg_024h_3_2.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_168h_2_1.trinity.bam | cut -f 1,3 > mlig_regen_reg_168h_2_1.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_H2long_000h_R2.transabyss.bam | cut -f 1,3 > mlig_pilot_stage_H2long_000h_R2.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_000h_3_1.soap.bam | cut -f 1,3 > mlig_regen_reg_000h_3_1.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_000h_3_2.oases.bam | cut -f 1,3 > mlig_regen_reg_000h_3_2.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_reg_072h_R1.soap.bam | cut -f 1,3 > mlig_pilot_reg_072h_R1.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_reg_012h_R2.trinity.bam | cut -f 1,3 > mlig_pilot_reg_012h_R2.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_024h_2_1.reassembly.bam | cut -f 1,3 > mlig_regen_reg_024h_2_1.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_120h_2_1.oases.bam | cut -f 1,3 > mlig_regen_control_120h_2_1.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_H2long_000h_R2.soap.bam | cut -f 1,3 > mlig_pilot_stage_H2long_000h_R2.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_024h_3_1.soap.bam | cut -f 1,3 > mlig_regen_reg_024h_3_1.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_072h_2_1.transabyss.bam | cut -f 1,3 > mlig_regen_reg_072h_2_1.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_A1350_000h_R2.reassembly.bam | cut -f 1,3 > mlig_pilot_stage_A1350_000h_R2.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_096h_1_1.trinity.bam | cut -f 1,3 > mlig_regen_reg_096h_1_1.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_012h_1_1.trinity.bam | cut -f 1,3 > mlig_regen_control_012h_1_1.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_A1_000h_R2.trinity.bam | cut -f 1,3 > mlig_pilot_stage_A1_000h_R2.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_000h_3_2.reassembly.bam | cut -f 1,3 > mlig_regen_reg_000h_3_2.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_096h_2_1.transabyss.bam | cut -f 1,3 > mlig_regen_control_096h_2_1.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_H1_000h_R1.trinity.bam | cut -f 1,3 > mlig_pilot_stage_H1_000h_R1.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_072h_2_1.transabyss.bam | cut -f 1,3 > mlig_regen_control_072h_2_1.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_024h_3_1.oases.bam | cut -f 1,3 > mlig_regen_control_024h_3_1.oases.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_000h_3_2.soap.bam | cut -f 1,3 > mlig_regen_control_000h_3_2.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_048h_3_1.transabyss.bam | cut -f 1,3 > mlig_regen_reg_048h_3_1.transabyss.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_A1350_000h_R2.soap.bam | cut -f 1,3 > mlig_pilot_stage_A1350_000h_R2.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_stage_A1_000h_R1.trinity.bam | cut -f 1,3 > mlig_pilot_stage_A1_000h_R1.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_024h_1_1.trinity.bam | cut -f 1,3 > mlig_regen_control_024h_1_1.trinity.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_pilot_reg_024h_R1.soap.bam | cut -f 1,3 > mlig_pilot_reg_024h_R1.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_reg_168h_3_1.reassembly.bam | cut -f 1,3 > mlig_regen_reg_168h_3_1.reassembly.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_012h_1_1.soap.bam | cut -f 1,3 > mlig_regen_control_012h_1_1.soap.data\n",
        "samtools view -q 42 /home/share/projects/mlig_transcriptome/assemblyAnalaysis/map/mlig_regen_control_048h_3_2.soap.bam | cut -f 1,3 > mlig_regen_control_048h_3_2.soap.data\n",
        "\n",
        "</textarea>"
       ],
       "metadata": {},
       "output_type": "pyout",
       "prompt_number": 4,
       "text": [
        "<IPython.core.display.HTML at 0x51a1c90>"
       ]
      }
     ],
     "prompt_number": 4
    },
    {
     "cell_type": "code",
     "collapsed": false,
     "input": [
      "from collections import defaultdict\n",
      "\n",
      "prefix = '/home/share/projects/mlig_transcriptome/read_cluster/'\n",
      "\n",
      "fs = list([x for x in os.listdir(prefix) if x.split('.')[-1] == 'data'])\n",
      "\n",
      "groups = defaultdict(list)\n",
      "for f in fs:\n",
      "    lib = f.split('.')[0]\n",
      "    groups[lib].append(f)\n",
      "\n",
      "def getReadIndex(mapFiles):\n",
      "    reads = defaultdict(list)\n",
      "    for mapFile in mapFiles:\n",
      "        inFile = open(prefix + mapFile)\n",
      "        for line in inFile:\n",
      "            meta = line.strip().split()\n",
      "            reads[meta[0]].append(meta[1])\n",
      "            \n",
      "    return reads"
     ],
     "language": "python",
     "metadata": {},
     "outputs": [],
     "prompt_number": 34
    },
    {
     "cell_type": "code",
     "collapsed": false,
     "input": [
      "'''\n",
      "cluster = defaultdict(int)\n",
      "for lib, mapFiles in groups.items():\n",
      "    r = getReadIndex(mapFiles)\n",
      "    for rid, tids in r.items():\n",
      "        if len(tids) > 1:\n",
      "            tids.sort()\n",
      "            tids = tuple(tids)\n",
      "            cluster[tids] += 1\n",
      "    \n",
      "outFile = open('/home/share/projects/mlig_transcriptome/read_cluster/cluster/cluster.data','w')\n",
      "for members, count in cluster.items():\n",
      "    outFile.write(','.join(members) + '\\t' + str(count) + '\\n')\n",
      "outFile.close()\n",
      "'''"
     ],
     "language": "python",
     "metadata": {},
     "outputs": [],
     "prompt_number": 35
    },
    {
     "cell_type": "code",
     "collapsed": false,
     "input": [
      "clusterFile = open('/home/share/projects/mlig_transcriptome/read_cluster/cluster/cluster.data')\n",
      "clusters = []\n",
      "for line in clusterFile:\n",
      "    meta = line.strip().split()\n",
      "    members = meta[0].split(',')\n",
      "    clusters.append((members, int(meta[1])))"
     ],
     "language": "python",
     "metadata": {},
     "outputs": [],
     "prompt_number": 1
    },
    {
     "cell_type": "code",
     "collapsed": false,
     "input": [
      "clusters.sort(key = lambda x : x[1], reverse = True)\n",
      "clusters.sort(key = lambda x : len(x[0]), reverse = True)"
     ],
     "language": "python",
     "metadata": {},
     "outputs": [],
     "prompt_number": 2
    },
    {
     "cell_type": "code",
     "collapsed": false,
     "input": [
      "df_cluster = pd.DataFrame([[len(x[0]), x[1]] for x in clusters],columns=['members','weight'])"
     ],
     "language": "python",
     "metadata": {},
     "outputs": [],
     "prompt_number": 37
    },
    {
     "cell_type": "code",
     "collapsed": false,
     "input": [],
     "language": "python",
     "metadata": {},
     "outputs": []
    }
   ],
   "metadata": {}
  }
 ]
}